<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.45">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>for-later--microbiome</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="For later- microbiome_files/libs/clipboard/clipboard.min.js"></script>
<script src="For later- microbiome_files/libs/quarto-html/quarto.js"></script>
<script src="For later- microbiome_files/libs/quarto-html/popper.min.js"></script>
<script src="For later- microbiome_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="For later- microbiome_files/libs/quarto-html/anchor.min.js"></script>
<link href="For later- microbiome_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="For later- microbiome_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="For later- microbiome_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="For later- microbiome_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="For later- microbiome_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">


</head>

<body class="fullcontent">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content" id="quarto-document-content">




<section id="healthy-microbiome" class="level2">
<h2 class="anchored" data-anchor-id="healthy-microbiome">Healthy Microbiome</h2>
<p>2024-10 from <a href="https://www.gutmicrobiotaforhealth.com/are-we-getting-closer-to-defining-a-healthy-gut-microbiome-heres-what-recent-studies-show/">Gut Microbiome for Health</a></p>
<blockquote class="blockquote">
<p>Another new study by Liping Zhao and colleagues at the Rutger Center for Microbiome Analysis, along with international collaborators, has utilized artificial intelligence models for identifying a set of gut microorganisms that play a critical role in digestion, immune responses, and mental health. What is new with the analysis is that it is based on genome-specific analysis and database independence, and it is focused on stable gut microbiome interactions4.</p>
</blockquote>
<blockquote class="blockquote">
<p>The core microbiome’s structure includes two distinct groups of bacteria (i.e., the Foundation Guild and the Pathobiont Guild) that compete with each other as an indicator of health and help differentiate cases from controls across 15 diseases across three continents and predict immunotherapy outcomes. Stable interactions within gut microbiome members appear more relevant than the abundance of microorganisms. These findings open a new potential way to disease prediction and classification and manage microbiome-related diseases through specific interventions that target the core microbiome related to health4.</p>
</blockquote>
</section>
<section id="microbes-in-the-air" class="level2">
<h2 class="anchored" data-anchor-id="microbes-in-the-air">Microbes in the air</h2>
<p>An international team of researchers found hundreds of different genera of fungi and bacteria at up to 3,000 meters, including many human pathogens: Escherichia coli, Serratia marcescens, Prevotella melaninogenica, Staphylococcus epidermidis, Staphylococcus haemolyticus, Staphylococcus saprophyticus, Cutibacterium acnes, Clostridium difficile, Clostridium botulinum, Stenotrophomonas maltophilia, Shigella sonnei, Haemophillus parainfluenzae and Acinetobacter baumannii and health-relevant fungi such as Malassezia restricta, Malassezia globosa, Candida parapsilosis and Candida zeylanoides, Sarocladium kiliense, Cladosporium halotolerans, and Cladosporium herbarum.</p>
<p>https://www.pnas.org/doi/10.1073/pnas.2404191121</p>
</section>
<section id="diet-effectiveness-depends-on-microbes" class="level2">
<h2 class="anchored" data-anchor-id="diet-effectiveness-depends-on-microbes">Diet Effectiveness Depends on Microbes</h2>
<p>Stanford’s Michael Snyder and his lab found <a href="https://www.cell.com/cell-reports-medicine/fulltext/S2666-3791(22)00434-7">Distinct factors associated with short-term and long-term weight loss induced by low-fat or low-carbohydrate diet intervention</a></p>
<blockquote class="blockquote">
<p>Interestingly, we observe minimal dietary differences between those who succeeded in long-term weight loss and those who did not. Instead, proteomic and gut microbiota signatures significantly differ between these two groups at baseline.</p>
</blockquote>
</section>
<section id="database-of-food-microbes" class="level2">
<h2 class="anchored" data-anchor-id="database-of-food-microbes">Database of Food Microbes</h2>
<p>Unexplored microbial diversity from 2,500 food metagenomes and links with the human microbiome Carlino, NiccolòAlvarez-Ordóñez, Avelino et al. Cell, Volume 0, Issue 0</p>
<p>From <em>Cell</em> August 2024:</p>
<blockquote class="blockquote">
<p>Here, we present curatedFoodMetagenomicData (cFMD), an open-access resource that collects food-associated microbial data to support the use of metagenomics in food science. The current release comprises 2,533 food metagenomes with standardized metadata, 1,950 of them newly sequenced within the MASTER EU Consortium. We generated 10,112 prokaryotic and 787 eukaryotic metagenome-assembled genomes (MAGs) from food that were grouped into 1,036 prokaryotic and 108 eukaryotic species clusters, 320 of which resulted to be uncharacterized when compared with &gt;1 M existing genomes. We included these MAGs into our pipelines for sensitive taxonomic profiling and applied it to 19,833 human metagenomes, revealing species- and strain-level overlaps along the food-human axis.</p>
</blockquote>
</section>
<section id="microbes-in-your-bathroom" class="level2">
<h2 class="anchored" data-anchor-id="microbes-in-your-bathroom">Microbes in Your Bathroom</h2>
<p>Northwestern University scientists poured through the microbes in a typical bathroom showerhead and toothbrush to find these places harbor numerous unidentified organisms, with surprisingly little overlap among the different environments.</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="BathroomMicrobesShowerheadToothbrush.jpg" class="img-fluid figure-img"></p>
<figcaption>A toothbrush has more diversity than a showerhead</figcaption>
</figure>
</div>
<p>Front. Microbiomes, 08 October 2024 Sec. Environmental Microbiomes Volume 3 - 2024 | https://doi.org/10.3389/frmbi.2024.1396560</p>
<p><a href="https://www.frontiersin.org/journals/microbiomes/articles/10.3389/frmbi.2024.1396560/full">Phage communities in household-related biofilms correlate with bacterial hosts</a></p>
</section>
<section id="microbes-in-your-microwave-oven" class="level2">
<h2 class="anchored" data-anchor-id="microbes-in-your-microwave-oven">Microbes in Your Microwave Oven</h2>
<p><a href="https://www.nature.com/articles/d41586-024-02553-9">Your microwave oven has its own microbiome</a></p>
<blockquote class="blockquote">
<p>Alba Iglesias, a microbiologist at the University of Valencia in Spain, and her colleagues swabbed 30 microwave ovens A total of 101 bacterial strains grew in the cultures. The dominant ones belonged to the Bacillus, Micrococcus and Staphylococcus genera, which commonly live on human skin and surfaces that people frequently touch. Human-skin bacteria were present in all three types of microwave oven, but were more abundant in the household and shared-use appliances. A few bacteria types associated with food-borne illnesses, including Klebsiella and Brevundimonas, also grew in some of the cultures from household microwaves.</p>
</blockquote>
<p>Iglesias, A., Martínez, L., Torrent, D. &amp; Porcar, M. Front. Microbiol. https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2024.1395751/full (2024).</p>
</section>
<section id="microbes-help-animals-detect-magnetic-fields" class="level2">
<h2 class="anchored" data-anchor-id="microbes-help-animals-detect-magnetic-fields">Microbes Help Animals Detect Magnetic Fields</h2>
<p>University of Central Florida biologist Robert Fitak <a href="https://www.ucf.edu/news/ucf-biologist-continues-unraveling-mystery-of-magnetic-bacteria/">created a refined database of magnetic bacteria</a> speculating that it’s microbes that help turtles and other animals navigate.</p>
<p><a href="https://www.ucf.edu/news/animals-magnetic-sixth-sense-may-come-from-bacteria-new-paper-suggests/">Animals’ Magnetic ‘Sixth’ Sense May Come from Bacteria, New Paper Suggests</a></p>
<blockquote class="blockquote">
<p>Fitak found, for the first time, that magnetotactic bacteria are associated with many animals, including a penguin species, loggerhead sea turtles, bats and Atlantic right whales. For instance, Candidatus Magnetobacterium bavaricum regularly occurred in penguins and loggerhead sea turtles, while Magnetospirillum and Magnetococcus regularly occurred in the mammal species brown bats and Atlantic right whales.</p>
</blockquote>
<p>They speculate that the microbes might live somewhere in the nervous tissue of these animals.</p>
</section>
<section id="most-microbes-are-dormant" class="level2">
<h2 class="anchored" data-anchor-id="most-microbes-are-dormant">Most Microbes are Dormant</h2>
<p>An estimated 60% of all microbes are lying dormant at any given moment, ready to be switched back to life when conditions are right.<a href="#fn1" class="footnote-ref" id="fnref1" role="doc-noteref"><sup>1</sup></a></p>
<p>A natural protein, Balon, latches onto ribosomes to lock it in place and shut down the cell. Balon is found in around 20% of all microbes, and even more microbes might harbor genes that behave similarly.</p>
</section>
<section id="microbes-spread-through-dust-storms" class="level2">
<h2 class="anchored" data-anchor-id="microbes-spread-through-dust-storms">Microbes Spread Through Dust Storms</h2>
<p>In <a href="https://phys.org/news/2024-03-unseen-storms-bacteria-fungi-world.html">Unseen travelers: Dust storms may spread bacteria and fungi around the world</a></p>
<blockquote class="blockquote">
<p>Dr.&nbsp;Shankar Chellam, professor in the Zachry Department of Civil and Environmental Engineering and A.P. and Florence Wiley Professor III at Texas A&amp;M University, and his now former student Dr.&nbsp;Sourav Das have furthered previous research&nbsp;<a href="https://pubs.acs.org/doi/10.1021/acs.est.3c04158">to identify microorganisms that might have hitched a ride in the dust plumes across the Atlantic</a>. Dr.&nbsp;Daniel Spalink, assistant professor and director of S.M. Tracy Herbarium at Texas A&amp;M, helped analyze the biology and identify bacteria and fungi in the samples.</p>
</blockquote>
</section>
<section id="human-milk-as-a-treatment-for-gut-disease" class="level2">
<h2 class="anchored" data-anchor-id="human-milk-as-a-treatment-for-gut-disease">Human Milk as a Treatment for Gut Disease</h2>
<p>Human milk oligosaccharides (HMOs), the third most abundant component of human milk are not found in other kinds of milk, yet play a vital role in the care and feeding of <em>Bifidobacterium</em>.</p>
<p>Prolacta Bioscience is one company that makes a human-equivalent HMO intended to feed gut bacteria, and perhaps, solve numerous gut problems associated with dysfunctional microbiomes.</p>
<p>Seattle-based Intrinsic Medicine is currently running on clinical trial to use HMO-based drugs for treatment of Parkinsons Disease.</p>
<p>From <a href="https://www.statnews.com/2024/03/29/human-milk-oligosaccarides-gut-microbiome-immune-boost/">Stat News Mar 2024</a></p>
<blockquote class="blockquote">
<p>Biomilq, for instance, synthesizes milk using mammary cells for people who have difficulty breastfeeding. And adults have occasionally tried to tap those benefits via dietary or nutritional supplements — and even by obtaining colostrum, the first milk produced by mammals within the first 48 to 72 hours after birth. Companies such as Armra are marketing colostrum, albeit from a bovine source, in powder form.</p>
</blockquote>
<p><a href="https://www.biorxiv.org/content/10.1101/2024.02.01.577565v1">Longitudinal profiling of the microbiome at four body sites reveals core stability and individualized dynamics during health and disease</a></p>
<p>Mentions uBiome’s Melissa Agnello for processing some samples through uBiome’s lab.</p>
<p>Longitudinal profiling of the microbiome at four body sites reveals core stability and individualized dynamics during health and disease</p>
<p>Xin Zhou, Xiaotao Shen, Jethro S. Johnson, Daniel J. Spakowicz, Melissa Agnello, Wenyu Zhou, Monica Avina, Alexander Honkala, Faye Chleilat, Shirley Jingyi Chen, Kexin Cha, Shana Leopold, Chenchen Zhu, Lei Chen, Lin Lyu, Daniel Hornburg, Si Wu, Xinyue Zhang, Chao Jiang, Liuyiqi Jiang, Lihua Jiang, Ruiqi Jian, Andrew W. Brooks, Meng Wang, Kévin Contrepois, Peng Gao, Sophia Miryam Schüssler-Fiorenza Rose, Thi Dong Binh Tran, Hoan Nguyen, Alessandra Celli, Bo-Young Hong, Eddy J. Bautista, Yair Dorsett, Paula Kavathas, Yanjiao Zhou, Erica Sodergren, George M. Weinstock, Michael P. Snyder bioRxiv 2024.02.01.577565; doi: https://doi.org/10.1101/2024.02.01.57756</p>
</section>
<section id="behavior" class="level2">
<h2 class="anchored" data-anchor-id="behavior">Behavior</h2>
<section id="social-transmission-of-microbiomes" class="level3">
<h3 class="anchored" data-anchor-id="social-transmission-of-microbiomes">Social Transmission of Microbiomes</h3>
<p>A 2024 study by Yale Network Scientist Nicholas Christakis says <a href="https://www.nature.com/articles/d41586-024-03804-5">Your friends shape your microbiome — and so do their friends</a>. They mapped the social networks and microbiomes of almost 2000 people in rural Honduras villages and found—sure enough—that people who interact regularly have more similar microbes than those who don’t.</p>
<p><a href="https://www.nature.com/articles/s41586-024-08222-1">Full text</a>: Beghini, F., Pullman, J., Alexander, M. et al.&nbsp;Gut microbiome strain-sharing within isolated village social networks. Nature (2024). https://doi.org/10.1038/s41586-024-08222-1</p>
</section>
<section id="alcholism-and-gut-microbes" class="level3">
<h3 class="anchored" data-anchor-id="alcholism-and-gut-microbes">Alcholism and Gut Microbes</h3>
<p><a href="https://www.nature.com/articles/d41586-024-00667-8">Could the gut give rise to alcohol addiction?</a> asks <em>Nature</em>, describing work on Alcohol Use Disorder by Sophie Leclercq, a biomedical scientist at the Catholic University of Louvain in Brussel, and others.</p>
<blockquote class="blockquote">
<p>Leclercq thinks that 30–40% of cases of AUD might have a gut-related component that could be targeted for treatment. A key challenge is determining exactly which components to target — it is as yet unclear what constitutes a ‘good’ microbiome. Day’s analysis suggests that bacteria such as Lactobacillus, were in abundance in people with AUD, whereas Akkermansia and some others were low.</p>
</blockquote>
<blockquote class="blockquote">
<p>The gut bacteria Lactobacillus, for example, can produce GABA; Enterococcus can produce serotonin; and Bacillus can make dopamine. Short-chain fatty acids (SCFAs) released when dietary fibre is fermented by bacteria in the gut also have neuroactive properties.</p>
</blockquote>
<hr>
<p>Review article (2023): <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10146621/">Gut Microbiota in Anxiety and Depression: Unveiling the Relationships and Management Options</a></p>
<blockquote class="blockquote">
<p>A greater incidence of depression is substantially linked to a lower protein consumption than recommended. A 10% increase in protein consumption was shown to reduce the incidence of depression considerably in South Korea and in the United States. Several biological explanations have linked the intake of protein and depression. These theories are supported by the fact that tryptophan, an amino acid, is a precursor of serotonin.</p>
</blockquote>
</section>
</section>
<section id="beyond-microbes" class="level2">
<h2 class="anchored" data-anchor-id="beyond-microbes">Beyond Microbes</h2>
<p><a href="https://www.sciencealert.com/obelisks-entirely-new-class-of-life-has-been-found-in-the-human-digestive-system?utm_source=join1440&amp;utm_medium=email&amp;utm_placement=newsletter">‘Obelisks’: Entirely New Class of Life Has Been Found in The Human Digestive System</a> Stanford University biologist Ivan Zheludev searched millions of published genome and identified at least 30K different Obelisks that appeared in about 10% of the samples.</p>
<p>It’s RNA with only 1000 nucleotides.</p>
<p>Viroid-like colonists of human microbiomes Ivan N. Zheludev, Robert C. Edgar, Maria Jose Lopez-Galiano, Marcos de la Peña, Artem Babaian, Ami S. Bhatt, Andrew Z. Fire bioRxiv 2024.01.20.576352; doi: https://doi.org/10.1101/2024.01.20.576352</p>
</section>
<section id="diversity-and-immunity" class="level2">
<h2 class="anchored" data-anchor-id="diversity-and-immunity">Diversity and Immunity</h2>
</section>
<section id="statistically-modeling-for-health-status" class="level2">
<h2 class="anchored" data-anchor-id="statistically-modeling-for-health-status">Statistically Modeling for Health Status</h2>
<p>Zhu, J., Xie, H., Yang, Z. et al.&nbsp;Statistical modeling of gut microbiota for personalized health status monitoring. Microbiome 11, 184 (2023). https://doi.org/10.1186/s40168-023-01614-x</p>
<p><a href="https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-023-01614-x">Statistical modeling of gut microbiota for personalized health status monitoring</a></p>
<blockquote class="blockquote">
<p>We systematically developed a statistical monitoring diagram for personalized health status prediction and analysis. Our framework comprises three elements: (1) a statistical monitoring model was established, the health index was constructed, and the health boundary was defined; (2) healthy patterns were identified among healthy people and analyzed using contrast learning; (3) the contribution of each bacterium to the health index of the diseased population was analyzed. Furthermore, we investigated disease proximity using the contribution spectrum and discovered multiple multi-disease-related targets.</p>
</blockquote>
<p>via <a href="https://www.facebook.com/groups/TheGutClubStoolTestDiscussionGroup/posts/6633282866800134/">Ken Lassessen</a></p>
</section>
<section id="eye-disease-and-gut-microbiome" class="level2">
<h2 class="anchored" data-anchor-id="eye-disease-and-gut-microbiome">Eye Disease and Gut Microbiome</h2>
<p><a href="https://www.ucl.ac.uk/news/2024/feb/blindness-some-inherited-eye-diseases-may-be-caused-gut-bacteria">Some inherited eye diseases, including blindness</a> may be caused by gut bacteria that</p>
<blockquote class="blockquote">
<p>RB1 gene is key to controlling the integrity of the lower gastrointestinal tract, the first ever such observation. There, it combats pathogens and harmful bacteria by regulating what passes between the contents of the gut and the rest of the body.</p>
<p>The team found that when the gene has a particular mutation, dampening its expression (reducing its effect), these barriers in both the retina and the gut can be breached, enabling bacteria in the gut to move through the body and into the eye, leading to lesions in the retina that cause sight loss.</p>
</blockquote>
<p>The research was conducted on mice by Professor Richard Lee (UCL Institute of Ophthalmology and Moorfields Eye Hospital NHS Foundation Trust).</p>
<p>https://www.cell.com/cell/abstract/S0092-8674(24)00108-9</p>
</section>
<section id="skin-microbiome-and-attractiveness-to-mosquitoes" class="level2">
<h2 class="anchored" data-anchor-id="skin-microbiome-and-attractiveness-to-mosquitoes">Skin Microbiome and Attractiveness to Mosquitoes</h2>
<p><a href="https://bmcmicrobiol.biomedcentral.com/articles/10.1186/s12866-022-02502-4">Skin microbiome alters attractiveness to Anopheles mosquitoes</a></p>
<blockquote class="blockquote">
<p>Staphylococcus 2 ASVs are four times as abundant in the highly-attractive compared to poorly-attractive group.</p>
</blockquote>
<p>via <a href="https://www.axios.com/2023/06/04/why-mosquitoes-bite-blood?utm_source=join1440&amp;utm_medium=email&amp;utm_placement=newsletter">Axios</a></p>
</section>
<section id="ticks-and-alpha-gal" class="level2">
<h2 class="anchored" data-anchor-id="ticks-and-alpha-gal">Ticks and Alpha-Gal</h2>
<p>see <a href="./index.html">Ticks, Alpha-Gal, Neu5gc and more</a></p>
<p>The CDC <a href="https://www.cdc.gov/ticks/alpha-gal/index.html">warns about</a></p>
<blockquote class="blockquote">
<p>Alpha-gal syndrome (AGS) is a serious, potentially life-threatening allergic condition. AGS is also called alpha-gal allergy, red meat allergy, or tick bite meat allergy.</p>
</blockquote>
<p>does it have an association with microbes?</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1007%2Fs11882-019-0835-9/MediaObjects/11882_2019_835_Fig2_HTML.png" class="img-fluid figure-img"></p>
<figcaption>Figure 2: IgM and IgG antibodies are generated by continuous stimulation by the intestinal microbiome and probably also by food</figcaption>
</figure>
</div>
</section>
<section id="gut-brain" class="level2">
<h2 class="anchored" data-anchor-id="gut-brain">Gut-Brain</h2>
<p><a href="https://www.science.org/content/article/your-gut-directly-connected-your-brain-newly-discovered-neuron-circuit">Science Magazine</a></p>
<blockquote class="blockquote">
<p>a new study reveals the gut has a much more direct connection to the brain through a neural circuit that allows it to transmit signals in mere seconds.</p>
</blockquote>
</section>
<section id="paleo-humans" class="level2">
<h2 class="anchored" data-anchor-id="paleo-humans">Paleo Humans</h2>
<p><a href="https://www.science.org/doi/10.1126/science.adf5300">Natural products from reconstructed bacterial genomes of the Middle and Upper Paleolithic</a> from a German team that &gt; Now, a new study from an interdisciplinary team has taken important steps to understanding stone age bacteria by sequencing genomes recovered from ancient dental calculus. The hardened tartar preserved bacterial fragments on the teeth of 12 Neanderthals and 34 humans that had lived anywhere from 102,000 to 150 years ago. Formed from plaque, this calculus fossilized during these humans’ lifetime, trapping genetic fragments inside.</p>
<p>Discover Magazine on <a href="https://www.discovermagazine.com/the-sciences/what-were-stone-age-bacteria-like-we-now-have-the-first-clues?utm_source=join1440&amp;utm_medium=email&amp;utm_placement=newsletter">Bacterial DNA from ancient humans</a></p>
</section>
<section id="microbiome-uniqueness" class="level2">
<h2 class="anchored" data-anchor-id="microbiome-uniqueness">Microbiome uniqueness</h2>
<p>See https://pubmed.ncbi.nlm.nih.gov/30150716/</p>
<p>A Chinese study that found microbiome patterns that predict health in one province don’t work in another.</p>
<p><span class="citation" data-cites="he_regional_2018">@he_regional_2018</span></p>
<p>He Y, Wu W, Zheng HM, Li P, McDonald D, Sheng HF, Chen MX, Chen ZH, Ji GY, Zheng ZD, Mujagond P, Chen XJ, Rong ZH, Chen P, Lyu LY, Wang X, Wu CB, Yu N, Xu YJ, Yin J, Raes J, Knight R, Ma WJ, Zhou HW. Regional variation limits applications of healthy gut microbiome reference ranges and disease models. Nat Med. 2018 Oct;24(10):1532-1535. doi: 10.1038/s41591-018-0164-x. Epub 2018 Aug 27. Erratum in: Nat Med. 2018 Sep 24;: PMID: 30150716.</p>
<hr>
<p>https://www.nature.com/articles/s42255-021-00348-0</p>
<p>see <a href="https://www.evernote.com/shard/s7/nl/748304/abae11d4-6790-43ab-9776-faf611a24f04?title=ISB:%20Gut%20Microbiome%20Pattern%20Reflects%20Healthy%20Aging%20and%20Predicts%20Extended%20Survival%20in%20Humans">evernote</a></p>
<p><span class="citation" data-cites="wilmanski_gut_2021">@wilmanski_gut_2021</span></p>
</section>
<section id="methods" class="level2">
<h2 class="anchored" data-anchor-id="methods">Methods</h2>
<p><a href="https://www.researchgate.net/publication/372684035_Greengenes2_unifies_microbial_data_in_a_single_reference_tree">Greengenes2 unifies microbial data in a single reference tree</a></p>
<blockquote class="blockquote">
<p>Studies using 16S rRNA and shotgun metagenomics typically yield different results, usually attributed to PCR amplification biases. We introduce Greengenes2, a reference tree that unifies genomic and 16S rRNA databases in a consistent, integrated resource. By inserting sequences into a whole-genome phylogeny, we show that 16S rRNA and shotgun metagenomic data generated from the same samples agree in principal coordinates space, taxonomy and phenotype effect size when analyzed with the same tree.</p>
</blockquote>
</section>
<section id="mapping-the-capacity-of-a-single-subjects-microbiome-to-metabolize-hundreds-of-drugs" class="level2">
<h2 class="anchored" data-anchor-id="mapping-the-capacity-of-a-single-subjects-microbiome-to-metabolize-hundreds-of-drugs">Mapping the Capacity of a Single Subject’s Microbiome to Metabolize Hundreds of Drugs</h2>
<p>57 drugs that are transformed by the microbiome, with lots of variance from person to person.</p>
<p>https://www.cell.com/cell/fulltext/S0092-8674(20)30563-8#secsectitle0035</p>
<p><span class="citation" data-cites="javdan_personalized_2020">@javdan_personalized_2020</span></p>
</section>


<div id="quarto-appendix" class="default"><section id="footnotes" class="footnotes footnotes-end-of-document" role="doc-endnotes"><h2 class="anchored quarto-appendix-heading">Footnotes</h2>

<ol>
<li id="fn1"><p>see ‘Most Life on Earth Is Dormant, After Pulling an ‘Emergency Brake’’ in https://www.quantamagazine.org/most-life-on-earth-is-dormant-after-pulling-an-emergency-brake-20240605<a href="#fnref1" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
</ol>
</section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>