<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.280">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Personal Science Guide to the Microbiome - 10&nbsp; My Experiments</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href="./diy_microbiome.html" rel="next">
<link href="./Experiments/microbes_to_watch.html" rel="prev">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>


<link rel="stylesheet" href="css/normalize.css">
</head>

<body class="nav-sidebar floating">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
      <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./Experiments/microbes_to_watch.html">Do It Yourself</a></li><li class="breadcrumb-item"><a href="./Experiments.html"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">My Experiments</span></a></li></ol></nav>
      <a class="flex-grow-1" role="button" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
      </a>
      <button type="button" class="btn quarto-search-button" aria-label="Search" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal sidebar-navigation floating overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="./">Personal Science Guide to the Microbiome</a> 
        <div class="sidebar-tools-main">
    <div class="dropdown">
      <a href="" title="Download" id="quarto-navigation-tool-dropdown-0" class="quarto-navigation-tool dropdown-toggle px-1" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Download"><i class="bi bi-download"></i></a>
      <ul class="dropdown-menu" aria-labelledby="quarto-navigation-tool-dropdown-0">
          <li>
            <a class="dropdown-item sidebar-tools-main-item" href="./psm.epub">
              <i class="bi bi-bi-journal pe-1"></i>
            Download ePub
            </a>
          </li>
          <li>
            <a class="dropdown-item sidebar-tools-main-item" href="./psm.pdf">
              <i class="bi bi-bi-file-pdf pe-1"></i>
            Download PDF
            </a>
          </li>
          <li>
            <a class="dropdown-item sidebar-tools-main-item" href="./psm.docx">
              <i class="bi bi-bi-file-word pe-1"></i>
            Download Docx
            </a>
          </li>
      </ul>
    </div>
</div>
    </div>
      </div>
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Intro</span></span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true">
 <span class="menu-text">General Overview</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./intro.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Getting Started</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./personalscience.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">What is Personal Science?</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" aria-expanded="true">
 <span class="menu-text">Science</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./Science/microbesEverywhere.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Microbes everywhere</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./microbes.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Microbes and you</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./Science/microbes_in_you.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Microbes In You</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./science.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">The Science of the Microbiome</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" aria-expanded="true">
 <span class="menu-text">Technology</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-3" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./methods.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">Methods</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-4" aria-expanded="true">
 <span class="menu-text">Do It Yourself</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-4" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-4" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./Experiments/microbes_to_watch.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Microbes to Watch</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./Experiments.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">My Experiments</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./diy_microbiome.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">DIY Microbiome Testing</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./references.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">12</span>&nbsp; <span class="chapter-title">References</span></span></a>
  </div>
</li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#sec-experimentSummary" id="toc-sec-experimentSummary" class="nav-link active" data-scroll-target="#sec-experimentSummary"><span class="header-section-number">10.1</span> Summary of My Experiments</a>
  <ul class="collapse">
  <li><a href="#diversity" id="toc-diversity" class="nav-link" data-scroll-target="#diversity"><span class="header-section-number">10.1.1</span> Diversity</a></li>
  </ul></li>
  <li><a href="#sec-experimentKefir" id="toc-sec-experimentKefir" class="nav-link" data-scroll-target="#sec-experimentKefir"><span class="header-section-number">10.2</span> Kefir and the Microbiome</a></li>
  <li><a href="#sec-experimentKombucha" id="toc-sec-experimentKombucha" class="nav-link" data-scroll-target="#sec-experimentKombucha"><span class="header-section-number">10.3</span> Kombucha</a></li>
  <li><a href="#experimentProbiotics" id="toc-experimentProbiotics" class="nav-link" data-scroll-target="#experimentProbiotics"><span class="header-section-number">10.4</span> Do Probiotics Work?</a>
  <ul class="collapse">
  <li><a href="#vsl" id="toc-vsl" class="nav-link" data-scroll-target="#vsl"><span class="header-section-number">10.4.1</span> VSL</a></li>
  </ul></li>
  <li><a href="#startMouth" id="toc-startMouth" class="nav-link" data-scroll-target="#startMouth"><span class="header-section-number">10.5</span> My Oral Microbiome*</a></li>
  <li><a href="#experimentDentist" id="toc-experimentDentist" class="nav-link" data-scroll-target="#experimentDentist"><span class="header-section-number">10.6</span> Visiting the dentist</a></li>
  <li><a href="#conclusions" id="toc-conclusions" class="nav-link" data-scroll-target="#conclusions"><span class="header-section-number">10.7</span> Conclusions</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title"><span id="sec-experiments" class="quarto-section-identifier"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">My Experiments</span></span></h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>

<p>Microbe numbers shift daily in response to your environment, so a single sample won’t give much more than a brief snapshot at a single point in time. Here are some of the experiments I’ve tried on myself, in over 600 tests since 2014. What happens in your case?</p>
<section id="sec-experimentSummary" class="level2" data-number="10.1">
<h2 data-number="10.1" class="anchored" data-anchor-id="sec-experimentSummary"><span class="header-section-number">10.1</span> Summary of My Experiments</h2>
<p>During the period from 2014 through early 2019, I sequenced over 600 samples of my microbiome. Inspired by the experiment in a 2014 paper by David Lawrence<a href="#fn1" class="footnote-ref" id="fnref1" role="doc-noteref"><sup>1</sup></a>, during most of that time I also carefully tracked the food I ate, my sleep, and other variables like activity or location. Most of my near-daily samples were of my gut, but I also regularly tested my skin, nose, and mouth. Since I’m generally healthy, I didn’t have a specific goal in mind other than to try to understand better what these microbes are doing, so many of my tests were taken while undergoing simple experiments, like eating a specific type of food or visiting a new location. While not necessarily up to the rigorous standards of a formal scientific trial, these “n of 1” studies on myself helped me discover several new interesting facts about my own microbiome, many of which appear to contradict other published studies. In addition, hundreds of people sent me their own test results, letting me compare many different microbiomes. And of course, I also followed the latest developments in scientific publications and the general press as I eagerly tried to learn more.</p>
<p>What follows is a brief overview of some of the key things I learned.</p>
<ul>
<li>The microbiome is highly variable from day to day, often moving in ways that appear indistinguishable from random.</li>
<li>Broad trends <em>are</em> there if you look closely. I found many intriguing new results.</li>
<li>It <em>is</em> possible to change your microbiome in specific circumstances.</li>
<li>People’s microbiomes are frustratingly different from one another. A feature that seems to be true about one person may not apply to another.</li>
</ul>
<section id="diversity" class="level3" data-number="10.1.1">
<h3 data-number="10.1.1" class="anchored" data-anchor-id="diversity"><span class="header-section-number">10.1.1</span> Diversity</h3>
<p>The general consensus is that diversity is good: a greater variety of microbes ensures more resilience against the daily threat of invaders. Many people, after taking just one test, often feel either reassured that their diversity is “good” or disappointed that it’s “bad”. But I find that day-to-day variability is high enough that it’s almost never useful to use a single result. For example, here’s my diversity during a typical week: (<a href="#fig-plotOneWeekDiversity">Figure&nbsp;<span>10.1</span></a>))</p>
<div class="cell" data-hash="Experiments_cache/html/fig-plotOneWeekDiversity_f7dd5e344c83dcf4df3ad388909a7985">
<div class="cell-output-display">
<div id="fig-plotOneWeekDiversity" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="assets/images/Experiments_files/figure-html/fig-plotOneWeekDiversity-1.png" class="quarto-discovered-preview-image img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;10.1: Diversity changes significantly day-to-day.</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>If Monday were my only test, I may have been disappointed with my 1.83 score. Wait another day or two and, with no significant changes in diet, I was up to 2.29 – before plunging to 1.78 by the weekend. Moral: don’t take a single result too seriously.</p>
<p>To get a sense of how much diversity can vary over a year (<a href="#fig-plotOneYearDiversity">Figure&nbsp;<span>10.2</span></a>))</p>
<div class="cell" data-hash="Experiments_cache/html/fig-plotOneYearDiversity_48f15820e4cff91054209e6e66b0d6a0">
<div class="cell-output-display">
<div id="fig-plotOneYearDiversity" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="assets/images/Experiments_files/figure-html/fig-plotOneYearDiversity-1.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;10.2: Gut diversity varies day-to-day but holds to a recognizable range within a single individual</figcaption><p></p>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="sec-experimentKefir" class="level2" data-number="10.2">
<h2 data-number="10.2" class="anchored" data-anchor-id="sec-experimentKefir"><span class="header-section-number">10.2</span> Kefir and the Microbiome</h2>
<p>Everyone interested in the microbiome eventually has to check out kefir. Google the phrase “one of the most potent probiotic foods available” and you’ll find kefir in all the top results. A <a href="http://www.bbc.com/news/health-38800977">recent BBC documentary</a> that tested people after consuming different types of “gut-friendly” foods found it had by far the biggest effect. My interest piqued when, after my <a href="#sec-experimentKombucha">disappointment with kombucha</a>, I spoke with a man who happened to mention his good luck with kefir as a solution to his many gut issues. On a doctor’s recommendation, he tried kefir for a number of years with limited success, until — frustrated with the $3/day expense of buying it at Trader Joe’s — he began making it himself at home. “What a difference!” he claimed.</p>
<p>Did it work for me? Yes! I found a very noticeable change in my gut microbiome — the most significant I’ve seen among my many experiments. Look at my daily levels of <em>Leuconostoc</em>, a prodigious synthesizer of Vitamin K known to be found in kefir. (<a href="#fig-plotAmountsKefir">Figure&nbsp;<span>10.3</span></a>)</p>
<div class="cell" data-hash="Experiments_cache/html/fig-plotAmountsKefir_9076943e58792897cbd1bcb8fb2d3267">
<div class="cell-output-display">
<div id="fig-plotAmountsKefir" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="assets/images/Experiments_files/figure-html/fig-plotAmountsKefir-1.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;10.3: Levels of this key microbe jump suddenly when I drink kefir (blue dots)</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The blue dots in the chart are days when I drank kefir. Since I sample near-daily over the entire chart, we can see that both of these taxa suddenly appeared shortly after I began to consume kefir. I had almost none beforehand. Also note that the levels seem to dip when I skip drinking for a few days, such as during my business trips out of town in mid-March and another in early-April.</p>
<p>So apparently it has a big effect on the microbiome. What is this stuff anyway?</p>
<p>The first thing to know about kefir is the pronunciation. Say “Keh-FEAR”, with the accent on the second syllable, not “KEE-fur” or “kEH-fir”. The Russian origin of the term is a reminder of a time in the distant past when — it’s unclear exactly where or how — the first batch was prepared and then passed along, its microbial components shared from person to person until it reached today’s status as a popular drink you can buy in most grocery stores.</p>
<p>Making it at home brings more than just financial benefits. Commercially-purchased drinks are subject to unavoidable regulatory, shelf-life, and consistency contraints that matter for successful business, but not necessarily for nutrition. More importantly, if you believe like I do that microbes are highly-customized to our environments, making at home will ensure that the kefir is well-adapted to your own personal microbial environment. The batch that survives and thrives in your kitchen will have proven its ability to withstand whatever conditions you face there.</p>
<p>Making it yourself is surprisingly easy. It begins with a bundle of the component microbes, a cauliflower-shaped substance usually called the “grain” or “seed” that looks like <a href="#fig-kefir_image">Figure&nbsp;<span>10.4</span></a></p>
<div class="cell" data-hash="Experiments_cache/html/fig-kefir_image_cff58540a8fb640ba7ee34c2e3612d9f">
<div class="cell-output-display">
<div id="fig-kefir_image" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="assets/images/Experiments/assets/kefir_grain.jpg" class="img-fluid figure-img" width="2016"></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;10.4: A few of these cauliflower-shaped kefir grains will fermet a whole glass of milk</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Instruction books often tell you to be careful how you handle the grains, but I find them robust enough that I pick them up with my bare fingers. I drop them into a glass of milk left I leave sitting on the counter overnight and — voila! — twenty four hours later, the liquid has turned into kefir. Pull out the kefir grains from that glass, plop it into another, and you’re all set for tomorrow’s batch. Unlike yogurt, which requires heating and a stable temperature, kefir doesn’t appear to care how it’s handled, so long as you keep it at room temperature and can wait for twenty four hours. The reaction might vary by a few hours if the room is a bit colder or warmer, but otherwise I find it surprisingly consistent. Just set and forget.</p>
<p>I found that the only hard part is getting started. Once you have the grains, making more kefir is easy, but where do you get the grains in the first place? It’s supposedly possible to make them from scratch using a goat-hide bag filled with pasteurized milk and the intestinal flora of a sheep, but I haven’t tried that myself. I’m told it works so long as you shake every hour and maintain a constant temperature.</p>
<p>You can order some starter grains online for under $25, but for shipping purposes the manufacturers generally give them to you in a freeze-dried form that requires a week or so of preparation before the microbes are fully alive and kicking out drinkable quantities of kefir.</p>
<p>I got mine by asking around until I found a neighbor who had been brewing his own. Anyone who makes homemade kefir will be happy to give you some extra grains. The fermentation process causes the grains to multiply, and you will find yourself throwing them out regularly.</p>
<p>The grains themselves contain a combination of lactic acid bacteria (<em>Lactobacillus</em>, <em>Lactococcus</em>, <em>Leuconostoc</em>), acetic acid bacteria (<em>Acetobacter</em>), and yeast, clumped together with casein (milk proteins) and complex sugars in a type of carbohydrate molecule called <em>kefiran</em>. The nutritional content apparently varies depending on fermentation time and other factors, but there’s a lot of good stuff in there<a href="#fn2" class="footnote-ref" id="fnref2" role="doc-noteref"><sup>2</sup></a> (<a href="#fig-kefirOltesNutrition">Figure&nbsp;<span>10.5</span></a>).</p>
<div class="cell" data-hash="Experiments_cache/html/fig-kefirOltesNutrition_3855d72956dfb908471b924b5055d54a">
<div class="cell-output-display">
<div id="fig-kefirOltesNutrition" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="assets/images/Experiments/assets/kefir-oltes-nutrition.jpg" class="img-fluid figure-img" width="286"></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;10.5: Nutritional content of kefir. <span class="citation" data-cites="otles_kefir_2003">(Source: <a href="references.html#ref-otles_kefir_2003" role="doc-biblioref">Otles and Otles 2003</a>)</span></figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>A rigorous microbial analysis by an Irish lab <a href="#fn3" class="footnote-ref" id="fnref3" role="doc-noteref"><sup>3</sup></a> shows precisely which microbes are present in kefir at various stages in the fermentation process. This chart shows the composition of ordinary pasteurized milk as it changes from before adding kefir grains (time 0 at the bottom) until 24 hours have passed (top) and the milk has been transformed into just <em>Acetobacter</em>, <em>Lactobacillus</em>, and <em>Leuconostoc</em>.</p>
<div class="cell" data-fig.caption="Microbial makeup of a kefir sample" data-hash="Experiments_cache/html/unnamed-chunk-6_40fd794d413b453050a2d078c5e75878">
<div class="cell-output-display">
<p><img src="assets/images/Experiments/assets/kefir-microbial-makeup.jpg" class="img-fluid" width="458"></p>
</div>
</div>
<p>The uBiome test I used unfortunately can’t detect yeasts, so I don’t have an easy way to track the non-bacterial microbes in my kefir. But I <em>can</em> run the mixture through the same gene sequencing that I use for my other samples. I tested the kefir twice: once by simply dabbing the swab into the mixture that was waiting for me in the morning, and another swab from the same batch after removing the grain for an additional 24-hour “second ferment”. This is what I found when I sequenced the kefir from two different batches: (<a href="#tbl-kefirTable1">Table&nbsp;<span>10.1</span></a>)</p>
<div class="cell" data-hash="Experiments_cache/html/tbl-kefirTable1_ef1e27d1ac1976d48e7da296d7652c43">
<div class="cell-output-display">
<div id="tbl-kefirTable1" class="anchored">
<table class="table table-sm table-striped small">
<caption>Table&nbsp;10.1: Results from sequencing two distinct types of kefir (Genus)</caption>
<thead>
<tr class="header">
<th style="text-align: left;"></th>
<th style="text-align: right;">Kefir1</th>
<th style="text-align: right;">Kefir2</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Lactococcus</td>
<td style="text-align: right;">96.06</td>
<td style="text-align: right;">1.07</td>
</tr>
<tr class="even">
<td style="text-align: left;">Leuconostoc</td>
<td style="text-align: right;">3.02</td>
<td style="text-align: right;">0.06</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Lactobacillus</td>
<td style="text-align: right;">0.22</td>
<td style="text-align: right;">98.40</td>
</tr>
<tr class="even">
<td style="text-align: left;">Faecalibacterium</td>
<td style="text-align: right;">0.14</td>
<td style="text-align: right;">0.01</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Roseburia</td>
<td style="text-align: right;">0.06</td>
<td style="text-align: right;">0.00</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<p>These are the only taxa that met the 0.07% abundance criteria discussed previously. But even without that cutoff, the uBiome pipeline shows no <em>Acetobacter</em>, despite its prominence in the study shown above.</p>
<p>I wondered if this is simply due to the way uBiome labels the taxa that are found. Maybe the label <em>Acetobacter</em> just isn’t often assigned to uBiome samples. When I checked, I could find none in any of my own samples or of the hundreds of others that people have sent me. What’s more, none was reported in a large population study<a href="#fn4" class="footnote-ref" id="fnref4" role="doc-noteref"><sup>4</sup></a> either. So apparently it just doesn’t show up often in humans, though I wonder why it wouldn’t show up in the 16S sequencing of my kefir sample.</p>
<p>The answer, according to the uBiome scientist I talked to, is that <em>Acetobacter</em> is too similar to other genera for it to be accurately distinguished with a 16S test. So if we can’t see at the genus level, let’s look at a higher level, such as phylum. <a href="#tbl-kefirTable2">Table&nbsp;<span>10.2</span></a></p>
<div class="cell" data-hash="Experiments_cache/html/tbl-kefirTable2_9e93627fee265db49c9e218d621c0709">
<div class="cell-output-display">
<div id="tbl-kefirTable2" class="anchored">
<table class="table table-sm table-striped small">
<caption>Table&nbsp;10.2: Results from sequencing two distinct types of kefir (Phylum)</caption>
<thead>
<tr class="header">
<th style="text-align: left;"></th>
<th style="text-align: right;">Kefir1</th>
<th style="text-align: right;">Kefir2</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Firmicutes</td>
<td style="text-align: right;">99.75</td>
<td style="text-align: right;">99.57</td>
</tr>
<tr class="even">
<td style="text-align: left;">Bacteroidetes</td>
<td style="text-align: right;">0.12</td>
<td style="text-align: right;">0.06</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Proteobacteria</td>
<td style="text-align: right;">0.09</td>
<td style="text-align: right;">0.36</td>
</tr>
<tr class="even">
<td style="text-align: left;">Actinobacteria</td>
<td style="text-align: right;">0.03</td>
<td style="text-align: right;">0.01</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Verrucomicrobia</td>
<td style="text-align: right;">0.01</td>
<td style="text-align: right;">0.00</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<p>Because <em>Acetobacter</em> is within Phylum <em>Proteobacteria</em> and Order <em>Rhodospirillales</em>, we would expect to see some of those microbes if any of it were present. Looks like my kefir doesn’t include anything remotely resembling <em>Acetobacter</em>.</p>
<p>That’s what’s in the kefir grain itself. How does regular drinking affect my gut microbiome?</p>
<p>To find any taxa that may have suddenly changed as a result of kefir-drinking, let’s look at a heatplot that shows the relative abundances of all my top microbes over time. Darker spots are days when I have <em>less</em> of a particular bacterium, lighter spots are days when I have more.</p>
<div class="cell" data-hash="Experiments_cache/html/fig-kefirPlotHeat_5d3e75ec514f7e1c45b6ff09307dc868">
<div class="cell-output-display">
<div id="fig-kefirPlotHeat" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="assets/images/Experiments_files/figure-html/fig-kefirPlotHeat-1.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;10.6: Daily abundances of each microbe over time.</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Note the sudden appearance of the genus <em>Fusicatenibacter</em>. You rarely see such a dramatic and consistent change as a result of an experiment, but unfortunately, little is known about this genus. A member of the Clostria class of phylum Firmicutes, an internet search reveals little of interest. But it definitely appears in my samples after drinking kefir.</p>
<p>In fact, look how the levels appear to coincide precisely with the periods when I drink kefir:</p>
<div class="cell" data-hash="Experiments_cache/html/fig-kefirPlotBar_12d7386d5be34868b03d4f230638ca5f">
<div class="cell-output-display">
<div id="fig-kefirPlotBar" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="assets/images/Experiments_files/figure-html/fig-kefirPlotBar-1.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;10.7: Abundance of two important genera over time. Blue dots are days when I drank kefir</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>This is especially interesting because the only previous date when my gut saw any of this taxa was in December – on another occasion when I drank some kefir. In fact, <em>Fusicatenibacter</em> is such a strong predictor of kefir drinking that I can use it as a way to look back in time to see the samples when I drank some.</p>
<p>How common is <em>Fusicatenibacter</em> in gut microbiomes? Here’s a density plot look at a few hundred samples collected from other people.</p>
<div class="cell" data-hash="Experiments_cache/html/unnamed-chunk-11_e3a74834f9d22d4e7894ee74fdd0fc71">
<div class="cell-output-display">
<p><img src="assets/images/Experiments_files/figure-html/unnamed-chunk-11-1.png" class="img-fluid" width="672"></p>
</div>
</div>
<p>Although most people have none, it’s not unusual for people to have a few percentage points of <em>Fusicatenibacter</em> regardless of whether they regularly drink kefir.</p>
<p>But other than this clear change in my gut microbiome, did I notice any differences in health?</p>
<p>Here the answer is more ambiguous. As a healthy adult, I don’t have any particular “problems” I’m trying to solve. I remained healthy during the period of the experiment, so the kefir certainly doesn’t appear to have made anything worse. My sleep hasn’t substantially changed either, and although I’m generally pretty even-tempered, I didn’t notice any particular changes positive or negative in my mood either.</p>
<p>The one area where, subjectively, where I feel different is in my overall sense of energy. Although I can’t put my finger on anything quantitative, I <em>do</em> notice that I seem to be a little more energetic on days when I drink kefir. Measuring that more precisely may be a good followup test.</p>
</section>
<section id="sec-experimentKombucha" class="level2" data-number="10.3">
<h2 data-number="10.3" class="anchored" data-anchor-id="sec-experimentKombucha"><span class="header-section-number">10.3</span> Kombucha</h2>
<p>For healthy bacteria-rich drinks that affect the microbiome, many people immediately think of kombucha. Served chilled during the summer, it has a well-deserved reputation as a natural refreshing alternative to soft drinks. Despite its tangy, mildly sweet taste, it has a surprisingly low amount of sugar: only six grams in a serving<a href="#fn5" class="footnote-ref" id="fnref5" role="doc-noteref"><sup>5</sup></a>, compared to more than 20 grams in the same amount of orange juice or 39 grams in a can of Coke.</p>
<p>The sugar is missing because it’s been eaten by microbes, a complex blend of bacteria and yeast that convert regular tea (usually black, but also oolong or green tea) into a complex, flavorful beverage. The fermentation process is ideal for adding other ingredients for taste, so there is no end to the interesting flavors possible, giving rise to a highly competitive commercial market: U.S. supermarkets sold $180&nbsp;Million of the drinks in 2015.</p>
<p>Kombucha fermentation begins with a SCOBY, a “Symbiotic Colony of Bacteria and Yeast”, a pancake-sized disk-shaped gelitintous object also known as a “mother” or “mushroom”, which it sort of resembles. Despite the nickname, the only funji in the SCOBY are yeasts, combined with a complex blend of bacteria and other single-celled microbes from many parts of the tree of life. The different microbes need one another to produce the distinctive sweet and fizzy taste. Yeast cells convert sucrose into fructose and glucose and produce ethanol; the bacteria convert glucose into gluconic acid and fructose into acetic acid; caffeine from the tea stimulates the entire reaction, especially the production of cellulose by special strains of bacteria.<a href="#fn6" class="footnote-ref" id="fnref6" role="doc-noteref"><sup>6</sup></a></p>
<p>There have been <a href="http://www.kombu.de/benefits.htm">many anecdotal claims of the effect of kombucha on health</a>, purporting benefits ranging from better eyesight and thicker hair to cures for various diseases, though not everyone thinks it’s healthy. Even some alternative health experts, like Dr.&nbsp;Andrew Weil, <a href="http://www.nytimes.com/2010/03/25/fashion/25Tea.html">recommend against it</a>. Many of the claims for and against kombucha have been studied experimentally, in mice as well as humans, often with compelling results, but I’m unable to find any good data showing how it affects the microbiome.</p>
<p>So I tested it myself.</p>
<p>For seven days, from July 27 to August 2, I drank 48 ounces per day of commercially-purchased <a href="http://gtslivingfoods.com/">GT’s Gingerade Kombucha</a>. That’s three full bottles, or six servings a day for a week.</p>
<p>The key bacteria in the SCOBY are from phylum <em>Proteobacteria</em> and order <em>Rhodospirillales</em> of acetic- and gluconic-acid producting microbes that include genus <em>Gluconacetobacter</em>, closely related to <em>Acetobacter</em>, the key to the fermentation of vinegar. Thanks to the action of these microbes, kombucha is quite acidic, between 2.5 and 3.5 pH, almost as acidic as the 1.5 or 2.5 of a healthy stomach. These bacteria apparently don’t survive ingestion. They are rarely, if ever, found in human guts<a href="#fn7" class="footnote-ref" id="fnref7" role="doc-noteref"><sup>7</sup></a>, so whatever effect, if any, they have on the microbiome is indirect.</p>
<p>The label claims each bottle contains one billion organisms of two microbial species. The first, <em>Saccharomyces boulardii</em>, is a popular “healthy” microbe, well-studied and proven as a safe digestion aid. A close cousin of brewer’s yeast, its cell wall tends to stick to pathogens, which may account for its proven ability to prevent and fight diarrhea.<a href="#fn8" class="footnote-ref" id="fnref8" role="doc-noteref"><sup>8</sup></a>. Unfortunately, it is not a bacterium, and so won’t be detectable in my 16S-based microbiome tests.</p>
<p>The other added species <em>Bacillus coagulans</em> is often found in human guts, and should be easy to find. The specific one used in GT’s drinks is the patented <a href="http://www.ganedenbc30.com/ganedenbc30">Bacillus coagulans GBI-30, 6086</a>, a particularly hardy spore-forming microbe that can survive boiling and baking. Because it’s well-studied and safe, it’s a popular “probiotic” food additive and appears to have some beneficial effects on digestion.<a href="#fn9" class="footnote-ref" id="fnref9" role="doc-noteref"><sup>9</sup></a></p>
<p>I tested my gut microbiome each day, as well as my mouth and skin microbiome at intervals during the experiment and sure enough, the <em>Bacillus</em> shows up loud and clear. (<a href="#fig-experimentKombuchaPlotBacillus">Figure&nbsp;<span>10.8</span></a>)</p>
<div class="cell" data-hash="Experiments_cache/html/fig-experimentKombuchaPlotBacillus_d21a021d392bc9afff7bbda0590aa573">
<div class="cell-output-display">
<div id="fig-experimentKombuchaPlotBacillus" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="assets/images/Experiments_files/figure-html/fig-experimentKombuchaPlotBacillus-1.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;10.8: The blue line represents days I drank 6 servings of kombucha.</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>It took a few days of heavy kombucha drinking, but eventually those microbes became detectable. Given the known hardiness of <em>Bacillus</em>, this isn’t necessarily all that surprising. Still, it’s a nice confirmation that the test works; after all, in my hundreds of daily tests, I see this microbe only in the few days after drinking this brand of kombucha. But maybe the <em>Bacillus</em> just comes in and out, safely protected as a spore, without really influencing my microbiome. Can we see evidence the kombucha affected something else about my microbiome?</p>
<p><strong>Diversity doesn’t seem to change</strong> (<a href="#fig-experimentKombuchaDiversity">Figure&nbsp;<span>10.9</span></a>). I looked at the overall mixture of microbes and abundances using the Shannon diversity metric, commonly used by ecologists to tell measure the richness and variety in an environment. Don’t let the scale of this graph fool you: I set it narrowly to see precisely how diversity changes each day. A Shannon diversity change of a tenth of a point or so, as in this graph, is pretty trivial.</p>
<div class="cell" data-hash="Experiments_cache/html/fig-experimentKombuchaDiversity_8841786c8585b0e4546a1f4cd55f8dff">
<div class="cell-output-display">
<div id="fig-experimentKombuchaDiversity" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="assets/images/Experiments_files/figure-html/fig-experimentKombuchaDiversity-1.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;10.9: How my overall family-level diversity changes while drinking kombucha. I drank 6 full servings on each of the days marked with the blue line.</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Diversity had been climbing before the experiment began, so I don’t think we can lay that initial increase on kombucha. Incidentally, had I not been testing daily, I might be tempted to say diversity <em>decreased</em>. This is something that makes me skeptical of the results of many scientific studies: the microbiome flucutates so much day-to-day that what you see is very dependent on <em>when</em> you test. (By the way, note that the July 30 sample is missing, due to a failure in the lab processing.)</p>
<p>Let’s look at that order <em>Rhodospirillales</em> that contains the genus <em>Acetobacter</em> found in the SCOBY. (<a href="#fig-experimentKombuchaRhodo">Figure&nbsp;<span>10.10</span></a>)</p>
<div class="cell" data-hash="Experiments_cache/html/fig-experimentKombuchaRhodo_3986eb1b80f900df827a0562f87ef815">
<div class="cell-output-display">
<div id="fig-experimentKombuchaRhodo" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="assets/images/Experiments_files/figure-html/fig-experimentKombuchaRhodo-1.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;10.10: Abundance of microbes that include the genus Acetobacter found in the SCOBY</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>If we squint enough, we might credit that large spike with kombucha drinking. It’s possible, but then how would you explain the crash the following day, or the other apparent spikes in other parts of the chart? I conclude it’s probably a coincidence. More than likely, microbes like this from the SCOBY itself are not in the beverage anyway.</p>
<p>What about other microbes? Here is a heatmap showing the changes of the top 20 genus during my experiment (<a href="#fig-kombuchaExperimentPlots">Figure&nbsp;<span>10.11</span></a>)</p>
<div class="cell" data-hash="Experiments_cache/html/fig-kombuchaExperimentPlots_8767f7e4cb7dc1304d3981af19420cc6">
<div class="cell-output-display">
<div id="fig-kombuchaExperimentPlots" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="assets/images/Experiments_files/figure-html/fig-kombuchaExperimentPlots-1.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;10.11: Top 20 changes during my experiment</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>I don’t see any patterns. Usually, if the experiment causes a change, I’ll see an obvious streak from left to right somewhere in the heatmap, but I don’t see that.</p>
<p>Finally, let’s look at the levels of a few “probiotic” microbes, including the one listed on the label (<a href="#fig-kombuchaAllProbiotics">Figure&nbsp;<span>10.12</span></a>)</p>
<div class="cell" data-hash="Experiments_cache/html/fig-kombuchaAllProbiotics_511c632b2ad198737cc84d43e837f39a">
<div class="cell-output-display">
<div id="fig-kombuchaAllProbiotics" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="assets/images/Experiments_files/figure-html/fig-kombuchaAllProbiotics-1.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;10.12: Abundance of key ‘probiotic’ microbes while consuming kombucha.</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>While <em>Akkermansia</em> seems to rise near the end of the sequence, it’s hard to see any real patterns here.</p>
<p>For comparison, let’s look at a longer time horizon (<a href="#fig-kombuchaPlotLong">Figure&nbsp;<span>10.13</span></a>)</p>
<div class="cell" data-hash="Experiments_cache/html/fig-kombuchaPlotLong_4db23ae54408bff475001cebf8a532c8">
<div class="cell-output-display">
<div id="fig-kombuchaPlotLong" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="assets/images/Experiments_files/figure-html/fig-kombuchaPlotLong-1.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;10.13: Daily abundance of key microbes while drinking kombucha (blue lines). Blank regions are days when I have no data.</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Although we can’t positively credit kombucha for that spike in <em>Bacillus</em> during my experiment, it’s interesting that I had none of it in the weeks beforehand, and that it disappeared again in the weeks afterwards. I drink this brand of kombucha occasionally, and yes the same microbe shows up occasionally too, sometimes a few days afterwards.</p>
<p>In my years of testing, I rarely see <em>Bacillus</em> in my gut microbiome, but the few times when it <em>does</em> appear, there seems to be a relationship to drinking the same brand of kombucha a few days beforehand. There are also times when I drink kombucha and <em>don’t</em> detect this microbe, so the association isn’t perfect, but then again this was the only time I had so much of this brand all at once.</p>
<p>My conclusion: when consumed in large amounts, GT’s Gingerade Kombucha leaves new <em>Bacillus</em> microbes in my gut. Although they don’t appear to stick around permanently, the association is strong enough that I bet it works in you too. Other microbes, including so-called “probiotic” ones, don’t change much at all.</p>
<p>But don’t take my word for it. The full dataset and analysis tools are on Github: <a href="https://github.com/richardsprague/kombucha" class="uri">https://github.com/richardsprague/kombucha</a></p>
<p>There is much more analysis that can be done with this data. Some of the ideas you might try are:</p>
<ul>
<li>Study correlations among the taxa. Which ones are correlated, and which are not?</li>
<li>Which taxa appeared and/or disappeared during the experiment?</li>
<li>Is there a relationship between the microbes known to be present in kombucha and those in any of the gut results?</li>
<li>How do these results compare to <em>you</em> when you drink kombucha?</li>
</ul>
<p>Please study as much as you like, and let me know what you find!</p>
<p>P.S. The term “kombucha” is an unfortunate mistranslation of a Japanese word （昆布茶） that means “seaweed tea”. A fermented version of seaweed tea exists, but it has nothing to do with the drink described here.</p>
<!-- ```{r child="Experiments/experiment-travelMexico.Rmd"} -->
<!-- ``` -->
<!-- ```{r child="Experiments/experiment-travelChina.Rmd"} -->
<!-- ``` -->
<!-- ```{r child="Experiments/experiment-camping.Rmd"} -->
<!-- ``` -->
<!-- ```{r child="Experiments/experiment-activity.Rmd"} -->
<!-- # Does my microbiome change depending on how much exercise I get? -->
<!-- ``` -->
</section>
<section id="experimentProbiotics" class="level2" data-number="10.4">
<h2 data-number="10.4" class="anchored" data-anchor-id="experimentProbiotics"><span class="header-section-number">10.4</span> Do Probiotics Work?</h2>
<p>Probiotic supplements are a $55 billion business, with food and beverages accounting for almost 80% of that, according to an August 2021 report by <a href="https://www.grandviewresearch.com/industry-analysis/probiotics-market">Grandview Research</a>. With unregulated health claims that range from the benign (“helps digestion”) to the fantastic (“A miracle cure!”), do they make a significant difference in my own gut microbiome? I tested myself to find out.</p>
<p>Among unhealthy people, there is evidence that, under a doctor’s care, <a href="http://www.cochrane.org/CD004827/IBD_probiotics-prevention-antibiotic-associated-diarrhea-children">probiotics can help</a> with antibiotic-associated diarrhea and similar conditions in children or among people <a href="http://www.cochrane.org/CD006095/IBD_the-use-of-probiotics-to-prevent-c.-difficile-diarrhea-associated-with-antibiotic-use">recovering from <em>C.</em> difficile</a> infections.</p>
<p>On the other hand, a <a href="http://genomemedicine.biomedcentral.com/articles/10.1186/s13073-016-0300-5">recent scientific review</a> of all well-done studies of probiotics among healthy people couldn’t find evidence that probiotics made much difference compared to a placebo in randomized controlled trials. When the data-heavy web site FiveThirtyEight did a <a href="http://fivethirtyeight.com/features/gut-week-introduction/">week-long series</a> on Gut Science, including a detailed survey of <a href="http://fivethirtyeight.com/features/gut-week-do-probiotics-work-are-they-good-for-me/">what’s known about probiotics</a>, they concluded: “There’s no evidence in humans, however, to support taking probiotics just to generally improve your gut health.”</p>
<p>A literature review by the US Agency for Healthcare Research and Quality found <a href="http://www.ahrq.gov/research/findings/evidence-based-reports/probiotsum.html">no safety issues in healthy adults</a>, but there is surprisingly little research to show that the pills actually do anything. The independent lab <a href="https://labdoor.com/rankings/probiotics">Labdoor</a> tests most common brands to see which actually contain the organisms claimed on the label, but I couldn’t find anyone who tests whether the body can absorb them or not. There have been a few peer-reviewed studies showing that <em>some</em> microbes in supplements can make it to the gut<a href="#fn10" class="footnote-ref" id="fnref10" role="doc-noteref"><sup>10</sup></a>, but these studies almost feel like special cases, where they try lots of microbes and only a few make it. It’s not clear that organisms in a typical off-the-shelf bottle of probiotics have ever been tested that way.</p>
<p>I’m especially interested in learning whether the probiotics in the supplements actually “stick” in my gut. Taking so many billion organisms in pill form all at once may very well show up in a single gut test result, but how do I know they’re not simply being flushed out of my system? Or worse, how do I know I’m not just crowding out something more important?</p>
<p>To find out, I tracked my microbiome daily while taking a high quality probiotic supplement, one that I received directly from the manufacturer. To be a fair test, one worth publicizing the brand name for better or worse, I’d want to try it out on multiple people, at multiple times. Because I didn’t do that this time, I won’t name the product other than to say that it’s from a “good” brand and well-recommended.</p>
<p>I took the supplement once per day for nine days. I would have continued for an even ten, but I was starting to feel uncomfortably bloated those last few days. While that’s an encouraging sign that the pill is working, I didn’t want to do anything to seriously mess up my gut. I’m doing this experiment for fun, and it won’t be fun if I get sick as a result.</p>
<p>Let’s look for at the overall abundances for the two genera that were in the supplements: <em>Bifidobacterium</em> and <em>Lactobacillus</em>. (<a href="#fig-probioticPlotAmounts">Figure&nbsp;<span>10.14</span></a>).</p>
<div class="cell" data-hash="Experiments_cache/html/fig-probioticPlotAmounts_973a09ac8372e43d8f069920dc867a50">
<div class="cell-output-display">
<div id="fig-probioticPlotAmounts" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="assets/images/Experiments_files/figure-html/fig-probioticPlotAmounts-1.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;10.14: Percent abundance of key microbes (genus-level) found in the gut while taking a probiotic supplement.</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The red dots represents days when I took a gut sample after consuming the probiotic. Unfortunately, despite taking and submitting samples daily, several of my results just didn’t have enough reads to be useful. This chart shows only the days when I have a sample of at least 10,000 reads.</p>
<p>Even with that caveat, it’s hard to see clear-cut evidence that the pill had a significant effect. Yes, I have slightly more of those two taxa by the end of the experiment, but seriously, not <em>that</em> much more.</p>
<p>Let’s look at a longer time horizon (<a href="#fig-probioticPlotFacets">Figure&nbsp;<span>10.15</span></a>).</p>
<div class="cell" data-hash="Experiments_cache/html/fig-probioticPlotFacets_1d2bfeeb58a4ed866093bec870ffcaa3">
<div class="cell-output-display">
<div id="fig-probioticPlotFacets" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="assets/images/Experiments_files/figure-html/fig-probioticPlotFacets-1.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;10.15: Percent abundance of key gut microbes over a three month period after taking a probiotic supplement.</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Hmmmm, it seems the levels of those particular genera <em>did</em> increase a tiny bit at the end of the experiment, but there are plenty of other times on the chart where I see spikes too. In fact, the biggest increase happened in September when I was living it up in New Orleans, eating red beans and rice – and no probiotic pills.</p>
<p>Maybe my view of the microbe ecology, hoping to see results in only one or two genera, is too simplistic. We know that the gut is an ecosystem. If you add lots of one type of organism, maybe that affects the abundances and ratios of other microbes, all of whom are in constant competition with one another. Is there a way to tell <em>overall</em> how the microbes are changing?</p>
<p>Let’s apply an <em>ordination</em> analysis. Essentially this means we look at all the samples together and work out how different the samples are from one another, based on some “distance metric” that compares the abundances of specific microbes. If the abundances of two samples are roughly the same, or if they tend to rise and fall together, then we plot them next to each other, and vice versa if they are not well-correlated. There is a mathematical way to do this where we combine all these different correlations over and over and pick just the two that seem to matter the absolute most, which we’ll plot on a two-dimensional graph (<a href="#fig-probioticPlot">Figure&nbsp;<span>10.16</span></a>)</p>
<div class="cell" data-hash="Experiments_cache/html/fig-probioticPlot_ab27e2b090372a6730a15a7616bd7749">
<div class="cell-output-display">
<div id="fig-probioticPlot" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="assets/images/Experiments_files/figure-html/fig-probioticPlot-1.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;10.16: NMDS ordination (Bray-Curtis) of gut samples for ten months before and after taking probiotic supplements.</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Hmm… that looks pretty random to me.</p>
<section id="other-people" class="level4" data-number="10.4.0.1">
<h4 data-number="10.4.0.1" class="anchored" data-anchor-id="other-people"><span class="header-section-number">10.4.0.1</span> Other people</h4>
<p>Since doing this experiment on myself, I’ve spoken with numerous others who’ve tried something similar: take a gut test, then start some type of probiotic supplement, and finally take another followup test a few days or weeks later.</p>
<p>Here’s an example, “Jeremy”, a healthy man in his 50s took <a href="https://www.florahealth.com/us/products/super-bifido-plus-probiotic/">this probiotic supplement</a>: $42 for one month of pills:</p>
<div class="cell" data-hash="Experiments_cache/html/probioticsSuperBifidoLabel_c0526aef5e89647d38b3b1443fb7d883">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="assets/images/Experiments/assets/labelSuper-Bifido-Plus-Probiotic-30cap.jpg" class="img-fluid figure-img" width="200"></p>
<p></p><figcaption class="figure-caption">Super Bifido Plus Probiotics contains high amounts of live Bifidobacteria and Lactobacillus.</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>and here’s the high level result:</p>
<div class="cell" data-hash="Experiments_cache/html/unnamed-chunk-37_4f6e84eae7961796391cb8a19cb30cb9">
<div class="cell-output-display">
<p><img src="assets/images/Experiments_files/figure-html/unnamed-chunk-37-1.png" class="img-fluid" width="672"></p>
</div>
</div>
<p>Next let’s look just at the microbes reported to be in the probiotic pills. Jeremy has three samples of interest: (1) taken in mid-summer, a month before starting the probiotics, (2) right before the month of pills, and (3) after completing 30 days of faithful pill taking.</p>
<div class="cell" data-hash="Experiments_cache/html/unnamed-chunk-38_67ae43ae732c365e9b212f3377fa9289">
<div class="cell-output-display">
<p><img src="assets/images/Experiments_files/figure-html/unnamed-chunk-38-1.png" class="img-fluid" width="672"></p>
</div>
<div class="cell-output-display">
<p><img src="assets/images/Experiments_files/figure-html/unnamed-chunk-38-2.png" class="img-fluid" width="672"></p>
</div>
</div>
<p>So although we <em>do</em> see a slight increase in both taxa, it’s hard to pin it solely on the probiotics. After all, he was at even higher levels a month <em>before</em> starting the pills.</p>
<p>Also, looking more closely at the read counts, I see that the final sample had the lowest, about 36,000 reads versus the 80,000+ reads of the other samples. When dealing with low-abundance bacteria, this can matter, but it’s impossible to tell precisely how much. The bottom line is that it’s possible that the probiotics had no effect whatsoever, and even if there was an effect, it was probably quite slight.</p>
<p>In fact, probiotics appear to have less of an effect even than travel. Here’s “Kevin”, a European man who moved to the United States.</p>
<div class="cell" data-hash="Experiments_cache/html/unnamed-chunk-39_3810a53826f56a3bfce98ca414bd31d7">
<div class="cell-output-display">
<p><img src="assets/images/Experiments_files/figure-html/unnamed-chunk-39-1.png" class="img-fluid" width="672"></p>
</div>
</div>
<p>Notice how Kevin’s microbiome shifted dramatically a month after arriving in the US. Soon after that, he began taking a probiotic supplement, but his gut – while different – hasn’t shifted as much as it did from the international move.</p>
</section>
<section id="vsl" class="level3" data-number="10.4.1">
<h3 data-number="10.4.1" class="anchored" data-anchor-id="vsl"><span class="header-section-number">10.4.1</span> VSL</h3>
<p>The most tested probiotic is VSL#3, and recently a woman sent me her microbiome test results after taking <a href="https://www.optibacprobiotics.co.uk/shop/for-women">Optibac</a> for 4 days prior to her second test. (<a href="#fig-experimentProbioticsVSL">Figure&nbsp;<span>10.17</span></a>).</p>
<p>In this case the abundances of these microbes went up significantly. Is that a coincidence? Hard to tell from a single sample, but perhaps this probiotic is one that makes it through and shows up in the results.</p>
<div class="cell" data-hash="Experiments_cache/html/fig-experimentProbioticsVSL_e7e3d34bf0200a0264fa43dae23a4c0a">
<div class="cell-output-display">
<div id="fig-experimentProbioticsVSL" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="assets/images/Experiments_files/figure-html/fig-experimentProbioticsVSL-1.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;10.17: Change in key microbe levels after a course of VSL#3</figcaption><p></p>
</figure>
</div>
</div>
</div>
<section id="tentative-conclusions-and-next-steps" class="level4" data-number="10.4.1.1">
<h4 data-number="10.4.1.1" class="anchored" data-anchor-id="tentative-conclusions-and-next-steps"><span class="header-section-number">10.4.1.1</span> (Tentative) Conclusions and next steps</h4>
<p>It is very difficult to say with this analysis that the probiotics had any effect that is detectable by the uBiome Explorer test.</p>
<p>Further analysis required:</p>
<ul>
<li><p><em>Consider other statistical analysis</em>. Although the two strains contained in the probiotic pill don’t appear to cause a change in the gut microbiome results, are there other changes that can be detected statistically. Perhaps there are other taxa that show a significant change.</p></li>
<li><p><em>Other time horizons</em>. Maybe the changes don’t happen immediately. Although at a high level, there doesn’t appear to a noticable lag in the levels of the probiotic strains, perhaps a more sophisticated data transformation would uncover something.</p></li>
</ul>
<!-- ```{r child="Experiments/experiment-dentist.qmd"} -->
<!-- ``` -->
<!-- ```{r child="Experiments/experiment-colon.Rmd"} -->
<!-- ``` -->
<!-- ```{r child="Experiments/experiment-kombucha.Rmd"} -->
<!-- ``` -->
<!-- ```{r child="Experiments/experiment-sleep.Rmd"} -->
<!-- ``` -->
<!-- ```{r child="Experiments/experiment-food.Rmd"} -->
<!-- ``` -->
<!-- ```{r child="Experiments/experiment-dietChange.Rmd"} -->
<!-- ``` -->
<!-- ```{r child="Experiments/experiment-methods.Rmd"} -->
<!-- #How does collection method influence the result? -->
<!-- ``` -->
<!-- ```{r child="Experiments/experiment-sites/microbesInSkin.Rmd"} -->
<!-- # Microbes in the skin -->
<!-- ``` -->
</section>
</section>
</section>
<section id="startMouth" class="level2" data-number="10.5">
<h2 data-number="10.5" class="anchored" data-anchor-id="startMouth"><span class="header-section-number">10.5</span> My Oral Microbiome*</h2>
<p>The first place that microbes enter is also one of the richest and most variable environments in the body.</p>
<p>Scientists added an odorless compound from wine to a culture of known oral bacteria, and sure enough: the bacteria generated compounds that we can smell: terpenes, benzenic compounds and lipid derivatives. Each of us has a unique oral microbiome, and scientists were able to that that this inter-person variability is large enough to explain at least some of the differences in how each of us perceive a glass of wine.</p>
<p>What are the most important species in my mouth?</p>
<p>To microbes, your body looks like a hollow tube: skin on the outside, gut on the inside, and a mouth to allow passage between the two. Like purgatory, the mouth is a gatekeeper where new microbes wait before being whisked into the heavenly warm breeding grounds of the digestive system. But it’s no easy waiting room either - the mouth contains many highly-distinct eco-systems, each as different from one another as the Sahara desert is to the bottom of the ocean. Most microbiome and genetic tests ask you to swab the inside of the cheek - an easy, straightforward place teeming with bacteria, but the bacteria in the cheeks can be very different from those on the tongue or the lips. I tested them all one morning right after waking up.</p>
<div class="cell" data-hash="Experiments_cache/html/mouthshow_093a5ef2c3e315dd97c122b2e954400e">
<div class="cell-output-display">
<table class="table table-sm table-striped small">
<thead>
<tr class="header">
<th style="text-align: left;"></th>
<th style="text-align: right;">Lips</th>
<th style="text-align: right;">Tongue</th>
<th style="text-align: right;">Cheek (Right)</th>
<th style="text-align: right;">Cheek (Left)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Streptococcus</td>
<td style="text-align: right;">57.36</td>
<td style="text-align: right;">6.44</td>
<td style="text-align: right;">38.41</td>
<td style="text-align: right;">43.42</td>
</tr>
<tr class="even">
<td style="text-align: left;">Haemophilus</td>
<td style="text-align: right;">19.61</td>
<td style="text-align: right;">4.97</td>
<td style="text-align: right;">6.36</td>
<td style="text-align: right;">7.59</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Gemella</td>
<td style="text-align: right;">8.47</td>
<td style="text-align: right;">2.23</td>
<td style="text-align: right;">10.67</td>
<td style="text-align: right;">12.64</td>
</tr>
<tr class="even">
<td style="text-align: left;">Actinobacillus</td>
<td style="text-align: right;">3.07</td>
<td style="text-align: right;">0.26</td>
<td style="text-align: right;">3.91</td>
<td style="text-align: right;">3.32</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Veillonella</td>
<td style="text-align: right;">2.14</td>
<td style="text-align: right;">7.70</td>
<td style="text-align: right;">1.81</td>
<td style="text-align: right;">2.20</td>
</tr>
<tr class="even">
<td style="text-align: left;">Granulicatella</td>
<td style="text-align: right;">1.60</td>
<td style="text-align: right;">0.88</td>
<td style="text-align: right;">2.13</td>
<td style="text-align: right;">2.36</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Neisseria</td>
<td style="text-align: right;">1.47</td>
<td style="text-align: right;">14.91</td>
<td style="text-align: right;">7.51</td>
<td style="text-align: right;">4.12</td>
</tr>
<tr class="even">
<td style="text-align: left;">Fusobacterium</td>
<td style="text-align: right;">1.24</td>
<td style="text-align: right;">14.99</td>
<td style="text-align: right;">5.94</td>
<td style="text-align: right;">6.96</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Porphyromonas</td>
<td style="text-align: right;">1.10</td>
<td style="text-align: right;">3.37</td>
<td style="text-align: right;">3.56</td>
<td style="text-align: right;">2.78</td>
</tr>
<tr class="even">
<td style="text-align: left;">Rothia</td>
<td style="text-align: right;">0.90</td>
<td style="text-align: right;">0.21</td>
<td style="text-align: right;">6.49</td>
<td style="text-align: right;">2.83</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Actinomyces</td>
<td style="text-align: right;">0.69</td>
<td style="text-align: right;">2.26</td>
<td style="text-align: right;">2.90</td>
<td style="text-align: right;">2.11</td>
</tr>
<tr class="even">
<td style="text-align: left;">Prevotella</td>
<td style="text-align: right;">0.45</td>
<td style="text-align: right;">13.31</td>
<td style="text-align: right;">1.09</td>
<td style="text-align: right;">1.73</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Alloprevotella</td>
<td style="text-align: right;">0.42</td>
<td style="text-align: right;">2.76</td>
<td style="text-align: right;">0.46</td>
<td style="text-align: right;">0.36</td>
</tr>
<tr class="even">
<td style="text-align: left;">Leptotrichia</td>
<td style="text-align: right;">0.22</td>
<td style="text-align: right;">8.20</td>
<td style="text-align: right;">1.64</td>
<td style="text-align: right;">1.96</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Capnocytophaga</td>
<td style="text-align: right;">0.20</td>
<td style="text-align: right;">3.87</td>
<td style="text-align: right;">0.59</td>
<td style="text-align: right;">0.64</td>
</tr>
<tr class="even">
<td style="text-align: left;">Pasteurella</td>
<td style="text-align: right;">0.10</td>
<td style="text-align: right;">0.02</td>
<td style="text-align: right;">2.28</td>
<td style="text-align: right;">0.87</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Lachnoanaerobaculum</td>
<td style="text-align: right;">0.04</td>
<td style="text-align: right;">1.19</td>
<td style="text-align: right;">0.09</td>
<td style="text-align: right;">0.19</td>
</tr>
<tr class="even">
<td style="text-align: left;">Campylobacter</td>
<td style="text-align: right;">0.03</td>
<td style="text-align: right;">2.05</td>
<td style="text-align: right;">0.39</td>
<td style="text-align: right;">0.50</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Johnsonella</td>
<td style="text-align: right;">0.02</td>
<td style="text-align: right;">1.43</td>
<td style="text-align: right;">0.07</td>
<td style="text-align: right;">0.12</td>
</tr>
<tr class="even">
<td style="text-align: left;">Bacteroides</td>
<td style="text-align: right;">0.00</td>
<td style="text-align: right;">0.00</td>
<td style="text-align: right;">0.95</td>
<td style="text-align: right;">0.23</td>
</tr>
</tbody>
</table>
<p>Dramatic differences in the types of microbes in each part of the mouth.</p>
</div>
</div>
<p>While there is some variation in the cheeks, there is a dramatic difference between them and the lips or tongue. Also interesting is the way the lips are dominated by just three taxa that make up more than 85% of the total abundance. In Inverse Simpson terms, the lips are the least diverse, whereas the tongue is the most diverse.</p>
<div class="cell" data-hash="Experiments_cache/html/fig-mouth_diversity_afef917eb9af3459be32002525f91372">
<div class="cell-output-display">
<div id="fig-mouth_diversity" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="assets/images/Experiments_files/figure-html/fig-mouth_diversity-1.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;10.18: Even within the same mouth, a surprising variance in diversity.</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Regular testing of my microbiome often yields unexpected surprises, and this one has me stumped. Beginning in December 2016 and for no apparent reason, my mouth was colonized suddenly by a particular species of <em>Streptococcus</em> that had not been there before. Why? I’m not aware of any major lifestyle or other changes to cause this: same toothpaste, same living conditions. A few dietary experiments here and there, but nothing that coincides with these changes.</p>
<p>At the species level, I eliminated all samples with under 10,000 reads. We see something interesting: for no apparent reason, the species of <em>Streptococcus</em> detected in my mouth has changed. Suddenly, in December 2016 my mouth was colonized by a particular species that had not been there before. Why? I hadn’t done anything special; I’m not aware of any major lifestyle or other changes to cause this.</p>
<p>I confirmed with the lab that it’s not contamination. What’s especially odd is that I experienced a shift like this twice now in one year. After comfortably floating along with Species <em>Streptococcus sp. BS35a</em> for more than six months, suddenly in August the balance shifted again, this time to <em>Streptococcus sp. 11aTha1</em>. Will it shift again? Who knows?</p>
<div class="cell" data-hash="Experiments_cache/html/fig-mouth_passenger_975c5e30a4dee13fd05640551d46d686">
<div class="cell-output-display">
<div id="fig-mouth_passenger" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="assets/images/Experiments_files/figure-html/fig-mouth_passenger-1.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;10.19: Odd shifts in some streptococcus species over time.</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>I confirmed with the lab that it’s not contamination.</p>
<p>What’s <em>especially</em> odd is that I experienced a shift like this <em>twice</em> now in one year. After comfortably floating along with Species <em>Streptococcus sp. BS35a</em> for more than six months, suddenly in August the balance shifted again, this time to <em>Streptococcus sp. 11aTha1</em>. Will it shift again? Who knows?</p>
<p>Earth’s atmosphere was originally void of oxygen, a poisonous gas to the first, so-called “anaerobic” bacteria who thrived precisely because there was no oxygen. Over eons, as oxygen levels increased these microbes found places to hide: deep, dark pockets inside multicellular creatures who traded an oxygen-free interior for the abundant, exotic metabolites the microbes could synthesize. In humans, these bacterial safe-houses begin in the mouth, where the oxygen is low enough to keep the lights on for the anaerobes, while allowing occasional blooms for the aerobic bacteria that thrive whenever the mouth is open and they find fresh air.</p>
<p>Most of them do apparently need moisture: your salivary glands, strategically located in your cheeks and at the bottom of the mouth, churn out 1–2 liters of saliva per day.</p>
<p>The complexity of the mouth microbiome is compounded by the variety of surfaces, hard and soft, each with its own propensity to allow the formation of biofilms, tenacious clusters that protect microbes against invaders. On teeth, we call it dental plaque, a favorite protective breeding ground of <em>Streptococcus mutans</em>, the cavity-causing villain that, once established, is hard to dislodge. I’m fortunate that my mouth microbiome appears to have none of this and it’s true that I never have cavities. I’ve seen levels as high as 2% in some people, who have to visit the dentist no matter how much they brush.</p>
<p>Most microbes go down the hatch to the stomach, of course, but overly-aggressive tooth brushing or dental work can let a few can sneak into the bloodstream directly, where they can find their way to the lungs, the liver, or the heart, sometimes with deadly consequences. The “viridans” streptococci are one well-studied example: beneficial in the mouth, they outcompete other streptococcus enough to prevent strep throat, yet are the leading cause of heart valve infections if they make it into the bloodstream.</p>
<p>These mouth microbes have other interesting properties that may affect much more than we think. When scientists added an odorless compound from wine to a culture of known oral bacteria, the bacteria generated compounds that we can smell: terpenes, benzenic compounds and lipid derivatives.<a href="#fn11" class="footnote-ref" id="fnref11" role="doc-noteref"><sup>11</sup></a> Each of us has a unique oral microbiome, and the experimenters were able to show that this inter-person variability is large enough to explain at least some of the differences in how each of us perceive a glass of wine.</p>
<p>So what about these odd new ones that showed up in my mouth?</p>
<p>I’ve looked up their names in every reference I can think of, but have found nothing. That’s not too surprising: about a third of oral microbes are known only by their gene sequences.<a href="#fn12" class="footnote-ref" id="fnref12" role="doc-noteref"><sup>12</sup></a> The most satisfying answer from a microbiology expert I consulted is that these are likely to be “passenger microbes”, doing nothing in particular helpful or harmful.</p>
<p>In other words, like so many other microbes in our environment, they are just along for the ride.</p>
</section>
<section id="experimentDentist" class="level2" data-number="10.6">
<h2 data-number="10.6" class="anchored" data-anchor-id="experimentDentist"><span class="header-section-number">10.6</span> Visiting the dentist</h2>
<p>The gut biome is interesting enough, but bacteria colonize just about every part of the body, so here’s an experiment to measure the mouth bacteria and how the varieties shift after a visit to the dentist.</p>
<p>The mouth harbors its own unique ecology of bacteria, much of which is entirely unexplored. Scientists from the Forsyth Institute in Boston are at the cutting edge of the research, and have characterized many of the species found in their test subjects mouths, but widespread human trials are still years from producing the kinds of results we see from the gut biome research.</p>
<p>So far the research <em>is</em> clear that many cavities are associated with one nasty species: <em>Streptococcus Mutans</em>. This bug contains receptors that adhere to the surface of the tooth, creating a slimy biofilm where, under the right conditions they breed rapidly. Interestingly, just a few hundred bacterial cells is enough for it to begin its work, feeding on glucose to create a reaction that combines with the tooth enamel to form plaque. The ever-present lactic acid in the mouth, a critical component of pre-digestion, reacts with the plaque to remove calcium from the tooth, leaving small, ever deepening holes that will destroy the tooth unless the dentist intervenes with a filling.</p>
<p>I visit my dentist in April and October each year, and I measure my mouth biome before and after each April visit. Unfortunately I don’t have before/after for the October samples, but I <em>do</em> have a sample taken a few days <em>afterwards</em>. Let’s look at an overall heatmap picture of all the dentist-related mouth samples:</p>
<div class="cell" data-hash="Experiments_cache/html/loadDentist_268a99a6cbbff205d4ee58ff534e03a4">
<div class="cell-output-display">
<p><img src="assets/images/Experiments_files/figure-html/loadDentist-1.png" class="img-fluid" width="672"></p>
</div>
</div>
<p>These are five samples taken over the course of one year, when many things can change. Nevertheless, the differences between the before/after samples is strikingly obvious. Despite being sampled only a day apart, there is clearly a major shift in the mouth microbiome after a dentist appointment and cleaning.</p>
<p>Here’s a more numerical breakdown at the genus level of the top ten microbes and their abundances before and after first visit:</p>
<div class="cell" data-hash="Experiments_cache/html/mouthBeforeAfter1_bc01430c4ac1c90b72a0f7d2c6805f59">
<div class="cell-output-display">
<table class="table table-sm table-striped small">
<thead>
<tr class="header">
<th style="text-align: left;"></th>
<th style="text-align: right;">% before</th>
<th style="text-align: right;">% after</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Haemophilus</td>
<td style="text-align: right;">22.3237</td>
<td style="text-align: right;">23.0557</td>
</tr>
<tr class="even">
<td style="text-align: left;">Neisseria</td>
<td style="text-align: right;">17.9975</td>
<td style="text-align: right;">6.2514</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Streptococcus</td>
<td style="text-align: right;">13.4156</td>
<td style="text-align: right;">36.0979</td>
</tr>
<tr class="even">
<td style="text-align: left;">Prevotella</td>
<td style="text-align: right;">8.8309</td>
<td style="text-align: right;">3.9503</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Veillonella</td>
<td style="text-align: right;">6.5300</td>
<td style="text-align: right;">4.3380</td>
</tr>
<tr class="even">
<td style="text-align: left;">Porphyromonas</td>
<td style="text-align: right;">6.1201</td>
<td style="text-align: right;">3.6216</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Leptotrichia</td>
<td style="text-align: right;">4.1791</td>
<td style="text-align: right;">0.9552</td>
</tr>
<tr class="even">
<td style="text-align: left;">Fusobacterium</td>
<td style="text-align: right;">4.0991</td>
<td style="text-align: right;">2.5904</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Gemella</td>
<td style="text-align: right;">3.8491</td>
<td style="text-align: right;">9.7999</td>
</tr>
<tr class="even">
<td style="text-align: left;">Capnocytophaga</td>
<td style="text-align: right;">2.6722</td>
<td style="text-align: right;">0.8850</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>The most abundant taxa, <em>Haemophilus</em>, stays relatively stable, while abundances of the second two taxa <em>Streptococcus</em> and <em>Neissaria</em> seem to switch places. The other taxa in the top ten also seem to drop in abundance, except for <em>Streptococcus</em> and <em>Gamella</em>.</p>
<p>Anything in the genus <em>Capnocytophaga</em> is an opportunistic pathogen, so I say good riddance. Usually they’re fine, but if your immune system dips they can turn bad.</p>
<p>To understand more precisely what changed, let’s look more closely just at the ones that disappeared:</p>
<div class="cell" data-hash="Experiments_cache/html/mouthUnique_e04c64552db0afff1207496b7fdaab6b">
<div class="cell-output-display">
<table class="table table-sm table-striped small">
<thead>
<tr class="header">
<th style="text-align: left;"></th>
<th style="text-align: right;">% before</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Centipeda</td>
<td style="text-align: right;">0.0614</td>
</tr>
<tr class="even">
<td style="text-align: left;">Chryseobacterium</td>
<td style="text-align: right;">0.0385</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Bilophila</td>
<td style="text-align: right;">0.0057</td>
</tr>
<tr class="even">
<td style="text-align: left;">Bacteroides</td>
<td style="text-align: right;">0.0042</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Dialister</td>
<td style="text-align: right;">0.0028</td>
</tr>
<tr class="even">
<td style="text-align: left;">Akkermansia</td>
<td style="text-align: right;">0.0028</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Blautia</td>
<td style="text-align: right;">0.0028</td>
</tr>
<tr class="even">
<td style="text-align: left;">Stenotrophomonas</td>
<td style="text-align: right;">0.0028</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Mycobacterium</td>
<td style="text-align: right;">0.0028</td>
</tr>
<tr class="even">
<td style="text-align: left;">Delftia</td>
<td style="text-align: right;">0.0028</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>All of these unique microbes are of such tiny abundances that it’s hard to rule out simple contamination or other problems with the sampling. Still, it is interesting that there was nothing new (at the genus level) in the “after” sample that wasn’t in the “before”. This is consistent with the expectation that a dental cleaning would, if anything, tend to <em>remove</em> taxa rather than introduce any new ones.</p>
<p>But that was just for a single dental visit. What happened when I repeated the experiment the following year?</p>
<div class="cell" data-hash="Experiments_cache/html/mouthBeforeAfter2_dcf07118ef55f44154585c777cd7677f">
<div class="cell-output-display">
<table class="table table-sm table-striped small">
<thead>
<tr class="header">
<th style="text-align: left;"></th>
<th style="text-align: right;">% before</th>
<th style="text-align: right;">% after</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Streptococcus</td>
<td style="text-align: right;">45.8502</td>
<td style="text-align: right;">43.1782</td>
</tr>
<tr class="even">
<td style="text-align: left;">Actinobacillus</td>
<td style="text-align: right;">15.7059</td>
<td style="text-align: right;">8.5416</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Gemella</td>
<td style="text-align: right;">13.3460</td>
<td style="text-align: right;">13.3400</td>
</tr>
<tr class="even">
<td style="text-align: left;">Haemophilus</td>
<td style="text-align: right;">6.4258</td>
<td style="text-align: right;">8.7729</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Neisseria</td>
<td style="text-align: right;">6.2026</td>
<td style="text-align: right;">8.4911</td>
</tr>
<tr class="even">
<td style="text-align: left;">Granulicatella</td>
<td style="text-align: right;">2.3758</td>
<td style="text-align: right;">3.2374</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Veillonella</td>
<td style="text-align: right;">2.2163</td>
<td style="text-align: right;">4.2491</td>
</tr>
<tr class="even">
<td style="text-align: left;">Leptotrichia</td>
<td style="text-align: right;">1.3154</td>
<td style="text-align: right;">1.3874</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Porphyromonas</td>
<td style="text-align: right;">1.2118</td>
<td style="text-align: right;">2.5798</td>
</tr>
<tr class="even">
<td style="text-align: left;">Pasteurella</td>
<td style="text-align: right;">1.0204</td>
<td style="text-align: right;">0.8744</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Interestingly, this time my most abundant taxa is <em>Streptococcus</em>, instead of <em>Haemophilus</em>.</p>
<p>Like last time and as expected, I found no new taxa <em>after</em> the cleaning, but here are the genus-level items that disappeared, all at such low abundances that we should probably chalk them up to contamination or other errors that creep in unavoidably between the time I take the sample and when they show up in my results.</p>
<div class="cell" data-hash="Experiments_cache/html/beforeAfterDentist2_0407966a9b1e7083533ffeb931a92ff1">
<div class="cell-output-display">
<table class="table table-sm table-striped small">
<thead>
<tr class="header">
<th style="text-align: left;"></th>
<th style="text-align: right;">% before</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Moryella</td>
<td style="text-align: right;">0.0558</td>
</tr>
<tr class="even">
<td style="text-align: left;">Stenotrophomonas</td>
<td style="text-align: right;">0.0318</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Mycobacterium</td>
<td style="text-align: right;">0.0239</td>
</tr>
<tr class="even">
<td style="text-align: left;">Centipeda</td>
<td style="text-align: right;">0.0239</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Candidatus Saccharimonas</td>
<td style="text-align: right;">0.0159</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Let’s start with the genus level. How much <em>Streptococcus</em> has been in my mouth, and to the degree that we know at the species level, which types of species are there?</p>
<p>Hmm, lots of different species here. But what about the cavity-linked <em>S. Mutans</em>? It turns out that I <em>do</em> have a tiny bit, but in just one sample long ago. And sure enough, my dentist confirms that I have no cavities.</p>
<p>Keeping <em>S. Mutans</em> at bay is an important way that I’ll try to avoid cavities, so to continue the experiment, I’ll look at what I can do to manipulate the mouth biome, beyond what I eat and drink. A key part of that is how I brush my teeth.</p>
<p>Like most Americans, for years I brushed exclusively with one of the name brand toothpastes, usually Crest or Colgate. But looking more closely at the labels, I see two ingredients that will be of interest to my oral microbiome: triclosan<a href="#fn13" class="footnote-ref" id="fnref13" role="doc-noteref"><sup>13</sup></a> and sodium lauryl sulfate, both of which are are known to affect microbes. In addition, the fluoride in the paste works partly by making the tooth enamel more difficult for bacteria.</p>
<p>Pre-modern humans didn’t have toothpaste, and certainly not the antimicrobial kinds that have become popular only in the past generation. Of course, tooth decay was a painful reality for many of our ancestors as well, but there is good evidence that serious teeth problems didn’t begin until the widespread availability of sugar after the European immigration to America five hundred years ago. Skulls of humans before agriculture show almost no tooth decay. Wild animals, including primates like gorillas and chimpanzees get far fewer tooth problems than modern people, another clue that teeth brushing isn’t the whole story.</p>
<p>Could it be that a healthy mouth requires a healthy <em>diversity</em> of bacteria, including versions of <em>Stroptococcus</em> that out-compete the cavity-causing kinds? But toothpaste with triclosan and other anti-microbials are wide-spectrum: they don’t target just the “bad” cavity-causing organisms. They also kill other species needed for digestion, or to control bad breath.</p>
<p>To find out more about whether oral diversity is a good or bad thing, for my continuing mouth experiment I changed toothpaste. Rather than continue with one of the leading fluoride brands, I switched to a more “natural” brand whose ingredient list does <em>not</em> contain antimicrobials.</p>
<!-- ```{r child="Experiments/microbesInMouth.qmd"} -->
<!-- # Microbes in the mouth -->
<!-- ``` -->
<!-- ```{r child="Experiments/experiment-sites/microbesInNose.Rmd"} -->
<!-- # Microbes in the nose -->
<!-- ``` -->
<!-- ```{r child="Experiments/experiment-other.Rmd"} -->
<!-- ``` -->
</section>
<section id="conclusions" class="level2" data-number="10.7">
<h2 data-number="10.7" class="anchored" data-anchor-id="conclusions"><span class="header-section-number">10.7</span> Conclusions</h2>
<p>Are these results are meaningful enough to be worthy of further analysis? How representative were these samples of my microbiome at the time? I submit only a tiny swab to uBiome’s lab; are the bacteria distributed evenly enough that the size or location of the sample doesn’t matter? Would I get similar results if I submitted two tests from slightly different sites on the same sample?</p>
<p>The answers to all these questions are unclear, but while we need to take these concerns seriously, my experience over many samples is that the uBiome results are consistent enough that, yes, the conclusions are actionable as long as we keep the limitations in mind. Here’s why I think so:</p>
<p>First, my results are consistent with other “healthy omnivore” submissions that uBiome has received from others with diets and health histories similar to mine. I would be concerned if, for example, my firmicutes/bacteroidetes ratio were reversed.</p>
<p>Second, 16S technology has proven accuracy when identifying unique organisms, so I can generally trust information about the overall level of diversity regardless of specific proportions. Since diversity tends to correlate with health – and is manipulable based on what I eat – my own experience shows that the changes I see in these results go up and down consistently in the expected way. Even if specific points on my microbiome map are fuzzy, the overall landmarks appear to be solid.</p>


<div id="refs" class="references csl-bib-body hanging-indent" role="list" style="display: none">
<div id="ref-david_host_2014" class="csl-entry" role="listitem">
David, Lawrence A., Arne C. Materna, Jonathan Friedman, Maria I. Campos-Baptista, Matthew C. Blackburn, Allison Perrotta, Susan E. Erdman, and Eric J. Alm. 2014. <span>“Host Lifestyle Affects Human Microbiota on Daily Timescales.”</span> <em>Genome Biology</em> 15 (7): R89. <a href="https://doi.org/10.1186/gb-2014-15-7-r89">https://doi.org/10.1186/gb-2014-15-7-r89</a>.
</div>
<div id="ref-dommels_survival_2009" class="csl-entry" role="listitem">
Dommels, Y. E. M., R. A. Kemperman, Y. E. M. P. Zebregs, R. B. Draaisma, A. Jol, D. A. W. Wolvers, E. E. Vaughan, and R. Albers. 2009. <span>“Survival of <span>Lactobacillus</span> Reuteri <span>DSM</span> 17938 and <span>Lactobacillus</span> Rhamnosus <span>GG</span> in the <span>Human</span> <span>Gastrointestinal</span> <span>Tract</span> with <span>Daily</span> <span>Consumption</span> of a <span>Low</span>-<span>Fat</span> <span>Probiotic</span> <span>Spread</span>.”</span> <em>Applied and Environmental Microbiology</em> 75 (19): 6198–6204. <a href="https://doi.org/10.1128/AEM.01054-09">https://doi.org/10.1128/AEM.01054-09</a>.
</div>
<div id="ref-dufresne_tea_2000" class="csl-entry" role="listitem">
Dufresne, C., and E. Farnworth. 2000. <span>“Tea, <span>Kombucha</span>, and Health: A Review.”</span> <em>Food Research International</em> 33 (6): 409–21. <a href="https://doi.org/10.1016/S0963-9969(00)00067-3">https://doi.org/10.1016/S0963-9969(00)00067-3</a>.
</div>
<div id="ref-munoz-gonzalez_ability_2015" class="csl-entry" role="listitem">
Muñoz-González, Carolina, Carolina Cueva, M. Ángeles Pozo-Bayón, and M. Victoria Moreno-Arribas. 2015. <span>“Ability of Human Oral Microbiota to Produce Wine Odorant Aglycones from Odourless Grape Glycosidic Aroma Precursors.”</span> <em>Food Chemistry</em> 187 (November): 112–19. <a href="https://doi.org/10.1016/j.foodchem.2015.04.068">https://doi.org/10.1016/j.foodchem.2015.04.068</a>.
</div>
<div id="ref-otles_kefir_2003" class="csl-entry" role="listitem">
Otles, Semih, and Ozlem Cagindi Otles Semih. 2003. <span>“Kefir : <span>A</span> <span>Probiotic</span> <span>Dairy</span>-<span>Composition</span>, <span>Nutritional</span> and <span>Therapeutic</span> <span>Aspects</span>.”</span> <em>Pakistan Journal of Nutrition</em> 2 (2): 54–59. <a href="https://doi.org/10.3923/pjn.2003.54.59">https://doi.org/10.3923/pjn.2003.54.59</a>.
</div>
<div id="ref-walsh_microbial_2016" class="csl-entry" role="listitem">
Walsh, Aaron M., Fiona Crispie, Kieran Kilcawley, Orla O’Sullivan, Maurice G. O’Sullivan, Marcus J. Claesson, and Paul D. Cotter. 2016. <span>“Microbial <span>Succession</span> and <span>Flavor</span> <span>Production</span> in the <span>Fermented</span> <span>Dairy</span> <span>Beverage</span> <span>Kefir</span>.”</span> Edited by Rachel J. Dutton. <em>mSystems</em> 1 (5): e00052–16. <a href="https://doi.org/10.1128/mSystems.00052-16">https://doi.org/10.1128/mSystems.00052-16</a>.
</div>
<div id="ref-zhernakova_population-based_2016" class="csl-entry" role="listitem">
Zhernakova, Alexandra, Alexander Kurilshikov, Marc Jan Bonder, Ettje F. Tigchelaar, Melanie Schirmer, Tommi Vatanen, Zlatan Mujagic, et al. 2016. <span>“Population-Based Metagenomics Analysis Reveals Markers for Gut Microbiome Composition and Diversity.”</span> <em>Science</em> 352 (6285): 565–69. <a href="https://doi.org/10.1126/science.aad3369">https://doi.org/10.1126/science.aad3369</a>.
</div>
</div>
</section>
<section id="footnotes" class="footnotes footnotes-end-of-document" role="doc-endnotes">
<hr>
<ol>
<li id="fn1"><p><span class="citation" data-cites="david_host_2014">David et al. (<a href="references.html#ref-david_host_2014" role="doc-biblioref">2014</a>)</span><a href="#fnref1" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn2"><p><span class="citation" data-cites="otles_kefir_2003">Otles and Otles (<a href="references.html#ref-otles_kefir_2003" role="doc-biblioref">2003</a>)</span> http://files.cienciapatodos.webnode.pt/200000022-79ffe7af9e/Kefir.pdf<a href="#fnref2" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn3"><p><span class="citation" data-cites="walsh_microbial_2016">Walsh et al. (<a href="references.html#ref-walsh_microbial_2016" role="doc-biblioref">2016</a>)</span>. Also <a href="https://www.youtube.com/watch?v=uyibFCgXexA">a 2-minute Youtube presentation</a><a href="#fnref3" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn4"><p>See <span class="citation" data-cites="zhernakova_population-based_2016">Zhernakova et al. (<a href="references.html#ref-zhernakova_population-based_2016" role="doc-biblioref">2016</a>)</span><a href="#fnref4" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn5"><p>Though <a href="https://www.bevnet.com/news/2016/kombucha-study-raises-sugar-content-questions">one large test</a> by the BevNet industry trade site says the labels may under-report the real amount,and in 2017 a judge approved a settlement to a class action claiming misleading sugar content by one manufacturer<a href="#fnref5" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn6"><p>The best scientific review I know is in <span class="citation" data-cites="dufresne_tea_2000">Dufresne and Farnworth (<a href="references.html#ref-dufresne_tea_2000" role="doc-biblioref">2000</a>)</span><a href="#fnref6" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn7"><p>For a list of microbes that <em>are</em> found in the gut, see: http://www.raeslab.org/companion/vlaams-darmflora-project.html<a href="#fnref7" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn8"><p>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4542552/<a href="#fnref8" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn9"><p>https://www.asm.org/index.php/general-science-blog/item/6761-bottoms-up-discover-the-microbes-in-probiotic-drinks <a href="#fnref9" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn10"><p><em>L. reuteri DSM 17938</em> and <em>L. rhamnosus GG</em> in <span class="citation" data-cites="dommels_survival_2009">Dommels et al. (<a href="references.html#ref-dommels_survival_2009" role="doc-biblioref">2009</a>)</span><a href="#fnref10" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn11"><p><span class="citation" data-cites="munoz-gonzalez_ability_2015">Muñoz-González et al. (<a href="references.html#ref-munoz-gonzalez_ability_2015" role="doc-biblioref">2015</a>)</span><a href="#fnref11" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn12"><p>https://www.nature.com/articles/sj.bdj.2016.865.pdf<a href="#fnref12" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn13"><p>Note that Crest hasn’t used this in its products since 2014<a href="#fnref13" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
</ol>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="./Experiments/microbes_to_watch.html" class="pagination-link">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Microbes to Watch</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="./diy_microbiome.html" class="pagination-link">
        <span class="nav-page-text"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">DIY Microbiome Testing</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
      &nbsp;
    </div>   
    <div class="nav-footer-center">Copyright © Richard Sprague 2023</div>
    <div class="nav-footer-right">
      &nbsp;
    </div>
  </div>
</footer>



</body></html>