
## Best Academic Papers {#bestAcademicPapers}

If you're new to the microbiome and would like to dig into the academic papers that drive the field, here's the selection that I consider required reading.


Microbiome science is in its infancy, but its enormous potential makes it an environment rich in highly speculative research, often with results that are overturned rapidly with new discoveries. So before you read anything else, I encourage a peek at this 2014 Nature article by Harvard epidemiology professor William P. Hanage: [Microbiome science needs a healthy dose of skepticism](http://www.nature.com/news/microbiology-microbiome-science-needs-a-healthy-dose-of-scepticism-1.15730)



### Popular Topics

You will find several themes repeated regularly in the popular microbiome press

  


#### The Firmicutes/Bacteroidetes ratio

The most recent, well-respected review @walters_meta-analyses_2014 [  Walters, Xu, Knight 2014 ](http://onlinelibrary.wiley.com/doi/10.1016/j.febslet.2014.09.039/full) says flat-out:

> the ratio changes between normal and obese individuals are not statistically significant overall and therefore should not be considered a general feature distinguishing normal and obese human gut microbiota across populations.
> 
>   


Another study [(Finucane 2014)](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0084689) goes into deeper statistical detail to conclude the same thing.

  
  @walters_meta-analyses_2014


  Walters William A.,Xu Zech and Knight Rob(2014), Meta-analyses of human gut microbes associated with obesity and IBD, FEBS Letters, 588, doi: 10.1016/j.febslet.2014.09.039

  

  Finucane MM, Sharpton TJ, Laurent TJ, Pollard KS (2014) A Taxonomic Signature of Obesity in the Microbiome? Getting to the Guts of the Matter. PLoS ONE 9(1): e84689. doi: 10.1371/journal.pone.0084689

#### Obesity and the microbiome

Although it's exciting to think that an obesity cure might be found in the microbiome, the most recent reviews shows that it's more difficult than originally thought. Here's the best summary [Full Text (open)](http://mbio.asm.org/content/7/4/e01018-16)


    @sze_looking_2016 Sze, Marc A., and Patrick D. Schloss. “Looking for a Signal in the Noise: Revisiting Obesity and the Microbiome.” mBio 7, no. 4 (September 7, 2016): e01018-16. doi:10.1128/mBio.01018-16.

#### "We are only 10% human"

It's a number based on a guess dating from 1977, but finally updated in 2016:

  


> Our analysis updates the widely-cited 10:1 ratio, showing that the number of bacteria in our bodies is actually of the same order as the number of human cells. Indeed, the numbers are similar enough that each defecation event may flip the ratio to favor human cells over bacteria.
> 
>   

    Sender, R., Fuchs, S., & Milo, R. (2016). Revised Estimates for the Number of Human and Bacteria Cells in the Body. PLOS Biology, 14(8), e1002533. https://doi.org/10.1371/journal.pbio.1002533

#### Cure/cause obesity by FMT

Several studies in mice hint that an obese microbiome can be transferred to a skinny one and vice versa:

    Turnbaugh, P. J., Ley, R. E., Mahowald, M. A., Magrini, V., Mardis, E. R., & Gordon, J. I. (2006). An obesity-associated gut microbiome with increased capacity for energy harvest. Nature, 444(7122), 1027–31. http://doi.org/10.1038/nature05414

#### Moving Pictures

>Here, we present the largest human microbiota time series analysis to date, covering two individuals at four body sites over 396 timepoints.

    Caporaso, J. G., Lauber, C. L., Costello, E. K., Berg-Lyons, D., Gonzalez, A., Stombaugh, J., … Kelley, S. (2011). Moving pictures of the human microbiome. Genome Biology, 12(5), R50. http://doi.org/10.1186/gb-2011-12-5-r50

Don't miss the [30-second Youtube summary](https://www.youtube.com/watch?v=a49hToLcG8Y).

### Academic Papers

When you're ready to go to the original sources, be careful: there are tens of thousands of studies, many of them contradictory and quickly out of date. Here are the ones I think deserve to be read first.

  


  * Historic papers (HMG)
  * Population studies (enterotype, population studies)
  * Specific microbes (Akkermansia, Bifidobacterium, etc.)
  * Methods

### General Overview

A detailed technical review of how scientists study the microbiome, with an emphasis on how to judge the quality of results. This is a good overview for a smart person who wants an introduction to how we know what we know.



<details>
<summary>@tyler_analyzing_2014 ([Full Text](http://www.medscape.com/viewarticle/828715_3))</summary>
Tyler, Andrea D, Michelle I Smith, and Mark S Silverberg. “Analyzing the Human Microbiome: A ‘How To’ Guide for Physicians.” The American Journal of Gastroenterology 109, no. 7 (July 2014): 983–93. doi:10.1038/ajg.2014.73.
</details>

Here is another one:
    
    Young, Vincent B. “The Role of the Microbiome in Human Health and Disease: An Introduction for Clinicians.” BMJ, March 15, 2017, j831. doi:10.1136/bmj.j831.
    
    
### Microbes and Behavior

A 2019 summary of the links between microbes and psychiatry: @ameringen_gut_2019

    Ameringen, M., Turna, J., Patterson, B., Pipe, A., Mao, R. Q., Anglin, R., & Surette, M. G. (2019). The gut microbiome in psychiatry: A primer for clinicians. Depression and Anxiety. https://doi.org/10.1002/da.22936


### Historic Papers

The final paper describing conclusions of the Human Microbiome Project:

    
    
```Human, T., Project, M., & Figures, S. (2012). Structure, function and diversity of the healthy human microbiome. Nature, 486(7402), 207–14. http://doi.org/10.1038/nature11234```

### Self-tracking

Track as much as you can about two people for an entire year: their diet, physical activity, and microbiome; look for correlations. Conclusion: the microbiome is remarkably stable and quickly recovers to its baseline. The "Methods" section is especially interesting because it goes into detail on how to find interesting statistical results with such complicated data.

    
    David, L. A., Materna, A. C., Friedman, J., Campos-Baptista, M. I., Blackburn, M. C., Perrotta, A., … Alm, E. J. (2014). Host lifestyle affects human microbiota on daily timescales. Genome Biology, 15(7), R89. http://doi.org/10.1186/gb-2014-15-7-r89

### Diet

Looking for a good overview of studies that link various microbes to diet?

The following two papers are the best summaries:

<details>
  <summary>@scott_influence_2013</summary>
<p>Scott, K. P., Gratz, S. W., Sheridan, P. O., Flint, H. J., & Duncan, S. H. (2013). The influence of diet on the gut microbiota. Pharmacological Research, 69(1), 52–60. <http://doi.org/10.1016/j.phrs.2012.10.020></p>
</details>
<details>
  <summary>@portune_gut_2017</summary>
<p>Portune, Kevin J., Alfonso Benítez-Páez, Eva Maria Gomez Del Pulgar, Victor Cerrudo, and Yolanda Sanz. “Gut Microbiota, Diet and Obesity-Related Disorders - the Good, the Bad and the Future Challenges.” Molecular Nutrition & Food Research, June 2016. doi:10.1002/mnfr.201600252.</p>
</details>

    

Here's another one; see the supplements for details about which foods affect which bacteria.

<details>
  <summary>@david_diet_2014</summary>
<p>
```David, Lawrence A., Corinne F. Maurice, Rachel N. Carmody, David B. Gootenberg, Julie E. Button, Benjamin E. Wolfe, Alisha V. Ling, et al. “Diet Rapidly and Reproducibly Alters the Human Gut Microbiome.” Nature 505, no. 7484 (December 11, 2013): 559–63. doi:10.1038/nature12820.```</p>
</details>

### Population studies

The American Gut project citizen science survey of more than 10,000 microbiome samples,  published its results in 2018, finding very few clear associations between self-reported *anything* (sex, age, diet) and microbial diversity -- except one: people who self-reporting eating the most diverse numbers of plants had higher diversity than those who didn't.

@mcdonald_american_2018

    McDonald, D., Hyde, E., Debelius, J. W., Morton, J. T., Gonzalez, A., Ackermann, G., … Gunderson, B. (2018). American Gut: an Open Platform for Citizen Science Microbiome Research. MSystems, 3(3). https://doi.org/10.1128/mSystems.00031-18


### Enterotypes

The intriguing idea that there may be identifiable patterns in our microbiomes, called enterotypes, was proposed in this highly-cited paper, which includes a detailed methods supplement to show you how to compute it yourself:

  

    
    
    Arumugam, Manimozhiyan, Jeroen Raes, Eric Pelletier, Denis Le Paslier, Takuji Yamada, Daniel R. Mende, Gabriel R. Fernandes, et al. “Enterotypes of the Human Gut Microbiome.” Nature 473, no. 7346 (May 12, 2011): 174–80. doi:10.1038/nature09944.

The idea that identifiable enterotypes may exist has been viewed skeptically in follow-up work.

  


### Large population summaries

Twin studies help tease out the different effects of human and microbial DNA. This is a recent update to a study of 1,126 twin pairs:

  

    
    
    Goodrich, Julia K., Emily R. Davenport, Michelle Beaumont, Matthew A. Jackson, Rob Knight, Carole Ober, Tim D. Spector, Jordana T. Bell, Andrew G. Clark, and Ruth E. Ley. “Genetic Determinants of the Gut Microbiome in UK Twins.” Cell Host & Microbe 19, no. 5 (May 2016): 731–43. doi:10.1016/j.chom.2016.04.017.

Two excellent papers present a detailed analysis of the microbiomes and associated phenotypic information from several thousand healthy people in the Belgian Flemish Gut Flora Project (N = 1106) and the Dutch LifeLines-DEEP study (N = 1135).

  


Falony, G., M. Joossens, S. Vieira-Silva, J. Wang, Y. Darzi, K. Faust, A. Kurilshikov, et al. “Population-Level Analysis of Gut Microbiome Variation.” Science 352, no. 6285 (April 29, 2016): 560–64. doi:10.1126/science.aad3503.

  


Zhernakova, A., A. Kurilshikov, M. J. Bonder, E. F. Tigchelaar, M. Schirmer, T. Vatanen, Z. Mujagic, et al. “Population-Based Metagenomics Analysis Reveals Markers for Gut Microbiome Composition and Diversity.” Science 352, no. 6285 (April 29, 2016): 565–69. doi:10.1126/science.aad3369.

Be sure to study the supplemental materials, especially Supplement Table 11, which includes details of the specific microbes.

  


### Methods

A good overview of the current state of how microbiome analysis is performed, from the sample collection processing, to the data pipeline and final bioinformatics summaries. It includes references to the top platforms (e.g QIME, Mothur, PICRUSt) along with the various tradeoffs of each:

  

    
    
    Amato, Katherine R. “[An Introduction to Microbiome Analysis for Human Biology Applications](http://onlinelibrary.wiley.com/doi/10.1002/ajhb.22931/full): Amato.” American Journal of Human Biology, October 2016. doi:10.1002/ajhb.22931.

## Other Resources

Elizabeth Bik keeps an excellent [Microbiome Papers Collection](https://microbiomedigest.com/microbiome-papers-collection/human-microbiome-paper-collection/) of a few dozen classic academic papers.

and you'll find even more in @tyler_analyzing_2014, which is strongly recommended.

### Software

ANCOM (@mandal_analysis_2015) is an open source software tool^[The R code is here: http://www.niehs.nih.gov/research/resources/software/biostatistics/ancom/index.cfm] to help understand abundances.  

When we compare populations from one ecosystem (e.g. my results on Monday) with another (e.g. my results on Tuesday), there is a fundamental statistical sense in which the two populations are not comparable.

This paper gives the analogy of trying to compare two forests after capturing 100 animals in each: you count 20 bears in one and 30 in the other. There are statistical ways to say with confidence that the first forest is composed of 20% bears and the other 30%, but there is no way to conclude that the second forest has more bears without knowing the total number of animals in each.

A reliance on relative abundances (i.e. percentages) carries other, statistical, problems. For example, the Pearson correlation coefficient is difficult to interpret, since the sum-to-one characteristic of relative abundances requires mathematically that there be some negative correlations. If the numbers were absolute, you wouldn’t necessarily have negative correlations.






